%!TEX root = ../thesis.tex
% ******************************* Thesis Appendix A ****************************
\chapter{Supplementary Tables} 
\clearpage

\newpage\null\thispagestyle{empty}\newpage

\begin{comment}
    
\section{Additional results Chapter 2}

\begin{table}[ht!]
  \renewcommand{\arraystretch}{0.9} % More space between rows
  \centering
   \caption[\glslink{imc}{IMC} panel]{\gls{imc} panel}
  \label{tab:app_imc_panel}
  \begin{tabularx}{\textwidth}{x l C}
    \toprule
    \rowcolor{headerblue}
    \textbf{Isotopes} & \hspace{2em}\textbf{Marker (\textit{Gene})} & 
    \RaggedRight{\hspace{4em}\textbf{Cell Type}} \\
    \midrule
    141Pr & \hspace{2em}INS (\textit{Ins1, Ins2}) & \RaggedRight{\hspace{4em}$\beta$-cells} \\
    \midrule
    %\rowcolor{rowgray}
    142Nd & \hspace{2em}B220 & \RaggedRight{\hspace{4em}B-cells} \\
    \midrule
    143Nd & \hspace{2em}CD3 & \RaggedRight{\hspace{4em}T-cells} \\
    \midrule
    %\rowcolor{rowgray}
    144Nd & \hspace{2em}CD138 (\textit{Sdc1}) & \RaggedRight{\hspace{4em}Plasma cells} \\
    \midrule
    %\rowcolor{selectedrow}
    145Nd & \hspace{2em}CD8 & \RaggedRight{\hspace{4em}CD8\textsuperscript{+} T-cells} \\
    \midrule
    146Nd & \hspace{2em}CD31 (\textit{Pecam1}) & \RaggedRight{\hspace{4em}Endothelial cells} \\
    \midrule
    147Sm & \hspace{2em}CD16 (\textit{Fcgr3}) & \RaggedRight{\hspace{4em}Monocytes} \\
    \midrule
    149Sm & \hspace{2em}CD19 (\textit{Cd19}) & \RaggedRight{\hspace{4em}B-cells} \\
    \midrule
    150Nd & \hspace{2em}CD11b &  \\
    \midrule
    151Eu & \hspace{2em}CD44 (\textit{Cd44}) & \RaggedRight{\hspace{4em}Activation marker} \\
    \midrule
    153Eu & \hspace{2em}CD127 (\textit{Il7r}) & \RaggedRight{\hspace{4em}Memory marker}  \\
    \midrule
    154Sm & \hspace{2em}CD69 (\textit{Cd69}) & \RaggedRight{\hspace{4em}Activation marker}  \\
    \midrule
    158Gd & \hspace{2em}CRTH2 (\textit{Ptgdr2}) & \RaggedRight{\hspace{4em}\acrfullpl{ilc}} \\
    \midrule
  %\rowcolor{rowgray}
    159Tb & \hspace{2em}Eomes & \RaggedRight{\hspace{4em}CD4\textsuperscript{+} T-cells with cytotoxic activity} \\
    \midrule
    %\rowcolor{selectedrow}
    160Gd & \hspace{2em}Ly6G (\textit{Ly6g}) & \RaggedRight{\hspace{4em}Granulocytes} \\
    \midrule
    161Dy & \hspace{2em}CD4 (\textit{Cd4}) & \RaggedRight{\hspace{4em}CD4\textsuperscript{+} T-cells} \\
    \midrule
    162Dy & \hspace{2em}F4/80 (\textit{Adgre1}) & \RaggedRight{\hspace{4em}\multirow{2}{*}{Macrophages}} \\
    \cline{1-2}
    165Ho & \hspace{2em}\glslink{mhc2}{MHC-II} (\textit{H2-Ab1}) & \\
    \midrule
    166Er & \hspace{2em}NKX6-1 (\textit{Nkx6-1}) & \RaggedRight{\hspace{4em}$\beta$-cells} \\
    \midrule
    168Er & \hspace{2em}CD45 (\textit{Ptprc}) & \RaggedRight{\hspace{4em}Immune cells} \\
    \midrule
    169Tm & \hspace{2em}FOXP3 (\textit{Foxp3}) & \RaggedRight{\hspace{4em}T-regulatory cells (\gls{treg})}  \\
    \midrule
    170Er & \hspace{2em}KI67 (\textit{Mki67}) & \RaggedRight{\hspace{4em}Proliferation} \\
    \midrule
    171Yb & \hspace{2em}CD11c (\textit{Itgax}) & \RaggedRight{\hspace{4em}Dendritic cells} \\
    \midrule
    172Yb & \hspace{2em}CD56 (\textit{Ncam1}) & \RaggedRight{\hspace{4em}\acrfull{nk} cells} \\
    \midrule
    174Yb & \hspace{2em}\glslink{glp1}{GLP-1}R (\textit{Glp1r}) & \RaggedRight{\hspace{4em}$\beta$-cells} \\
    \midrule
    176Yb & \hspace{2em}Histone H3 & \RaggedRight{\hspace{4em}\multirow{3}{*}{\makecell{Nuclei/Cells}}} \\
    \cline{1-2}
    191Ir & \hspace{2em}\multirow{2}{*}{DNA Intercalator} & \\
    \cline{1-1}
    193Ir &  & \\
    \bottomrule
  \end{tabularx}
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[Summary of the generated \glslink{scr}{scRNA-seq} data]{Summary of the generated \gls{scr} data}
     \label{tab:app_scrna_meta}
  \begin{tabularx}{\textwidth}{x C C C C C C}
    \toprule
    \rowcolor{headerblue}
    \textbf{Cohort\textsuperscript{*}} & \textbf{Age of mice} & \textbf{Feeding-Time} & \textbf{Diet} & \textbf{Estimated number of cells$\dagger$} & \textbf{Median genes per cell} & \textbf{Median \gls{umi} per cell} \\
    
    % \makecell{\\\\Cohort\textsuperscript{*}} & \makecell{Age\\of mice \\ (in weeks)} & \makecell{Feeding\\Time \\ (in weeks)} & \makecell{\\\\Diet} & \makecell{\\\\\\Mean\\reads\\per cell} & \makecell{\\\\\\Median\\genes\\per cell} & \makecell{\\\\\\Median\\\gls{umi}\\per cell} \\
    
    \midrule
     W1-Chow-1 & \multirow{5}{*}{\makecell{10 wks}} & \multirow{5}{*}{\makecell{1 wk}} &  \multirow{3}{*}{\makecell{Chow}} & 3087 & 2987 & 13402 \\
     \cline{1-1} \cline{5-7}
     W1-Chow-2 &  &  &  & 18291 & 1818 & 6826 \\
     \cline{1-1} \cline{5-7}
     \makecell{W1-Chow-3\textsuperscript{**}} &  &  &  & 8419 & 3297 & 13461 \\
     \cline{1-1} \cline{4-4} \cline{5-7}
     W1-WD-1 &  &  & \multirow{2}{*}{\makecell{\gls{wd}}}  & 7881 & 3125 & 14433 \\
     \cline{1-1} \cline{5-7}
     W1-WD-2 &  &  &  & 28277 & 2381 & 9430 \\
    \midrule
    W12-Chow-1 & \multirow{5}{*}{\makecell{21 wks}} & \multirow{5}{*}{\makecell{12 wks}} &  \multirow{3}{*}{\makecell{Chow}} & 2989 & 2574 & 10448 \\
     \cline{1-1} \cline{5-7}
     W12-Chow-2 &  &  &  & 11641 & 3308 & 14677 \\
     \cline{1-1} \cline{5-7}
     \makecell{W12-Chow-3\textsuperscript{**}} &  &  &  & 6832 & 3676 & 15980 \\
     \cline{1-1} \cline{4-4} \cline{5-7}
     W12-WD-1 &  &  & \multirow{2}{*}{\makecell{\gls{wd}}}  & 7527 & 1790 & 5608 \\
     \cline{1-1} \cline{5-7}
     W12-WD-2 &  &  &  & 29151 & 2185 & 9499 \\
    \midrule
    Aging-1 & \multirow{2}{*}{\makecell{104 wks}} & \multirow{2}{*}{\makecell{95 wks}} &  \multirow{2}{*}{\makecell{Chow}} & 26170 & 1728 & 5478 \\
     \cline{1-1} \cline{5-7}
     Aging-2 &  &  &  & 26170 & 2532 & 10141 \\
    \bottomrule
    
  \end{tabularx}
  \vspace{0.1cm}
  \RaggedRight{\footnotesize \\ 
  \textsuperscript{*}\hspace{0.7em}Each cohort had $>$ 20 mice\\ 
  \textsuperscript{**} The number of cells post \gls{qc} is indicated in parantheses in \textbf{\autoref{fig:app_scrna_qc} A,B} \\ 
  $\dagger$\hspace{0.6em}No CD45 enrichment was performed}
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[Count statistics of macrophages across metabolic stress and aging]{Count statistics of macrophages across metabolic stress and aging}
     \label{tab:app_imc_macrophages}
  \begin{tabularx}{\textwidth}{xxCCCC}
    \toprule
    \rowcolor{headerblue}
    \textbf{Macrophage} & \textbf{Condition} & \textit{\textbf{N \glspl{roi}}} & \textit{\textbf{N \glspl{tma}}} & \textit{\textbf{N cells}} & \textbf{\% of CD45\textsuperscript{+} CD19\textsuperscript{-}} \\
    \midrule
    \multirow{7}{*}{\makecell{F4/80\textsuperscript{\textit{high}}}} & W1-Chow & 3 & 3 & 17 & 0.15 \\
    \cline{2-6}
    & W12-Chow & 3 & 3 & 26 & 0.23 \\
    \cline{2-6}
    & W24-Chow & 3 & 3 & 28 & 0.25 \\
    \cline{2-6}
    & W105-Chow & 6 & 3 & 11 & 0.1 \\
    \cline{2-6}
    & W1-WD & 2 & 2 & 14 & 0.12 \\
    \cline{2-6}
    & W12-WD & 3 & 3 & 3 & 0.03 \\
    \cline{2-6}
    & W24-WD & 3 & 3 & 198 & 1.75 \\
    \midrule
    \multirow{7}{*}{\makecell{F4/80\textsuperscript{\textit{low}}}} & W1-Chow & 7 & 7 & 18 & 0.16 \\
    \cline{2-6}
    & W12-Chow & 1 & 1 & 3 & 0.03 \\
    \cline{2-6}
    & W24-Chow & 5 & 5 & 42 & 0.37 \\
    \cline{2-6}
    & W105-Chow & 9 & 3 & 48 & 0.42 \\
    \cline{2-6}
    & W1-WD & 1 & 1 & 9 & 0.08 \\
    \cline{2-6}
    & W12-WD & 4 & 4 & 17 & 0.15 \\
    \cline{2-6}
    & W24-WD & 8 & 8 & 658 & 5.82 \\
    \midrule
    \multirow{7}{*}{\makecell{F4/80\textsuperscript{\textit{-}}}} & W1-Chow & 10 & 10 & 192 & 1.70 \\
    \cline{2-6}
    & W12-Chow & 6 & 6 & 37 & 0.33 \\
    \cline{2-6}
    & W24-Chow & 9 & 9 & 161 & 1.43 \\
    \cline{2-6}
    & W105-Chow & 8 & 3 & 88 & 0.78 \\
    \cline{2-6}
    & W1-WD & 10 & 10 & 59 & 0.52 \\
    \cline{2-6}
    & W12-WD & 12 & 11 & 134 & 1.19 \\
    \cline{2-6}
    & W24-WD & 12 & 12 & 559 & 4.95 \\
    \bottomrule
    
  \end{tabularx}
  \vspace{0.1cm}
   \RaggedRight{\footnotesize \\ 
   \textbf{\textit{N \glspl{roi}}}, Number of \glspl{roi} that contain the macrophage\\ 
   \textbf{\textit{N \glspl{tma}}}, Number of \glspl{tma} that contain the macrophage\\
   \textbf{\textit{N cells}}, Number of cells of the macrophage per condition} 
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[Marker genes for islet macrophages from \glslink{scr}{scRNA-seq}]{Marker genes for islet macrophages depicted in \textbf{\autoref{fig:chp2_scrna_macrophages} B}}
     \label{tab:app_scrna_macrophages_markers}
  \begin{tabularx}{\textwidth}{xxCCC}
    \toprule
    \rowcolor{headerblue}
    \textbf{Cluster} & \textbf{Gene} & \textbf{Avg. log\textsubscript{2}FC} & \textbf{p-value} & \textbf{Adj. p-value} \\
    \midrule
    \multirow{3}{*}{\makecell{Macs-1}} & \textit{Gm42418} & 0.842914251 &  \num{1.23056e-82}  & \num{2.83633e-78} \\
    \cline{2-5}
     & \textit{Txnip} & 0.63129266 & \num{2.0868e-228} & \num{4.8098E-224} \\
     \cline{2-5}
     & \textit{Apoe} & 0.386001623 & \num{1.3098e-138} & \num{3.0189e-134} \\
     \midrule

     \multirow{4}{*}{\makecell{Macs-2}} & \textit{Ifnb1} & 1.827282475 & 0 & 0 \\
    \cline{2-5}
     & \textit{Cxcl10} & 1.381994858 & 0 & 0 \\
     \cline{2-5}
     & \textit{Ifit1} & 1.107384537 & 0 & 0 \\
     \cline{2-5}
     & \textit{Tnf} & 1.102939153 & 0 & 0 \\
     \midrule

      \multirow{5}{*}{\makecell{Macs-3}} & \textit{Ccl5} & 2.302441975 & \num{5.2708e-136} & \num{1.2149e-131} \\
    \cline{2-5}
     & \textit{Ifitm3} & 2.043965 & 0 & 0 \\
     \cline{2-5}
     & \textit{Cxcl9} & 2.011294836 & \num{6.5682e-268} & \num{1.5139e-263} \\
     \cline{2-5}
     & \textit{Irf7} & 1.561541749 & 0 & 0 \\
     \cline{2-5}
     & \textit{Ifit3} & 1.514048212 & 0 & 0 \\
     \midrule
     \multirow{3}{*}{\makecell{Macs-4}} & \textit{Stmn1} & 3.142720712 &  0  & 0 \\
    \cline{2-5}
     & \textit{Tubb5} & 2.158930706 & \num{1.7046e-289} & \num{3.9289e-285} \\
     \cline{2-5}
     & \textit{Birc5} & 1.76888341 & 0 & 0 \\
     \midrule
     \multirow{3}{*}{\makecell{Macs-5}} & \textit{Ctsd} & 2.697777377 &  \num{9.6438e-295}  & \num{2.2228e-290} \\
    \cline{2-5}
     & \textit{Prdx1} & 2.304428257 & 0 & 0 \\
     \cline{2-5}
     & \textit{Lgals3} & 2.239307655 & 0 & 0 \\
     \bottomrule
  \end{tabularx}
  \vspace{0.1cm}
   \RaggedRight{\footnotesize \\
    \textbf{\textit{Avg. log\textsubscript{2} FC}}, log fold-change of the average expression of the gene across the sub-populations\\ 
   \textbf{\textit{Adj. p-value}}, adjusted p-value based on Bonferroni correction using all genes in the dataset} 
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[\glslink{de}{DE} genes for Macs-2 sub-population from \glslink{scr}{scRNA-seq}]{\gls{de} genes for Macs-2 noted in \textbf{\autoref{fig:app_scrna_macrophages_macs2_dge}}}
     \label{tab:app_scrna_macs2_dge}
  \begin{tabularx}{\textwidth}{xxCCC}
    \toprule
    \rowcolor{headerblue}
    \textbf{Comparison} & \textbf{Gene} & \textbf{Avg. log\textsubscript{2}FC} & \textbf{p-value} & \textbf{Adj. p-value} \\
    \midrule
    \multirow{13}{*}{\makecell{W1\\WD vs Chow}} & \textit{Gm42418} & 2.783864318 &	\num{5.7232E-100} &	\num{9.70804E-97} \\
    \cline{2-5}
     & \textit{Rsad2} & 1.834361533 &	\num{3.98455E-44} &	\num{5.63233E-42} \\
     \cline{2-5}
     & \textit{Ccl5} & 1.610340545 &	\num{2.03523E-48} & \num{4.06147E-46}	 \\
     \cline{2-5}
     & \textit{Ifnb1} & 1.088404496 &	\num{4.0512E-35} &	\num{3.43593E-33} \\
     \cline{2-5}
     & \textit{H2-M2} & 1.043453796 &	\num{7.23315E-23} &	\num{2.583E-21} \\
     \cline{2-5}
     & \textit{Klf2} & 1.032178458 & \num{9.35952E-32} & \num{6.04803E-30}	 \\
     \cline{2-5}
     & \textit{Ifit1} & 0.958978009 &	\num{8.50347E-27} & \num{3.95178E-25}	 \\
     \cline{2-5}
     & \textit{Ccl3} & 0.88102183 &	\num{2.27966E-45} & \num{3.51533E-43}	 \\
     \cline{2-5}
     & \textit{Ccrl2} &	0.737455955 &	\num{3.76732E-49} & \num{7.74584E-47}	 \\
     \cline{2-5}
     & \textit{Cxcl10} & 0.702056725 &	\num{3.6256E-26} & \num{1.60782E-24} \\
     \cline{2-5}
     & \textit{Il1a} & 0.548736797 &	\num{2.23955E-09} &	\num{2.09014E-08} \\
     \cline{2-5}
     & \textit{Tnf} &	0.536009454 &	\num{2.9506E-28} &	\num{1.49401E-26} \\
     \cline{2-5}
     & \textit{Socs3} & 0.51300086 &	\num{5.22905E-15} &	\num{1.02838E-13} \\
     
     \midrule

     \multirow{9}{*}{\makecell{W12\\WD vs Chow}} & \textit{Ifit1} & 1.166817047 & \num{2.13288E-11} & \num{2.67873E-09} \\
    \cline{2-5}
     & \textit{Ifit2} &	1.088208435 & \num{2.95605E-12} & \num{4.77332E-10} \\
     \cline{2-5}
     & \textit{Rsad2} &	1.012233307 &	\num{7.30873E-08} & \num{3.59187E-06} \\
     \cline{2-5}
     & \textit{Ifnb1} &	0.863915536 & \num{3.01818E-08} & \num{1.66417E-06} \\
     \cline{2-5}
     & \textit{Ccl5} &	0.810882495 & \num{2.3049E-07} & \num{9.59009E-06}	 \\
     \cline{2-5}
     & \textit{Tnf} & 0.776475499 & \num{2.18764E-16} & \num{9.89107E-14} \\
     \cline{2-5}
     & \textit{Cxcl10} & 0.756659835 & \num{1.59187E-12} & \num{2.91785E-10} \\
     \cline{2-5}
     & \textit{Ccl3} & 0.706779016 &\num{5.08884E-14} &	\num{1.50055E-11} \\
     \cline{2-5}
     & \textit{Ccl4} & 0.55250498 & \num{1.33822E-12} & \num{2.66936E-10}	 \\
     
     \midrule

      \multirow{13}{*}{\makecell{Chow\\Aging vs Non-aging}} & \textit{Gm42418} &	2.091748135 &	\num{1.81878E-45} & \num{1.27096E-41}	 \\
    \cline{2-5}
     & \textit{H2-M2} &	0.86555075 &	\num{1.63242E-14} &	\num{9.50611E-12} \\
     \cline{2-5}
     & \textit{Klf2} &	0.65680418 & \num{1.84206E-06} &	0.00018389 \\
     \cline{2-5}
     & \textit{Cd14} &	0.492994005 & \num{8.02353E-08} & \num{1.41684E-05}		 \\
     \cline{2-5}
     & \textit{Dnajb1} &	0.456117077 & 	\num{1.87533E-06} &	0.000184575	 \\
     \cline{2-5}
     & \textit{Wfdc17} & 0.43422713 &	\num{9.70386E-06} &	0.000671392 \\
     \cline{2-5}
     & \textit{Apoe} &	0.423200038 &	\num{1.63801E-08} &	\num{3.577E-06} \\
     \cline{2-5}
     & \textit{Itm2b} &	0.419068447 & \num{7.18664E-24} & \num{2.09013E-20}	 \\
     \cline{2-5}
     & \textit{Rhob} &	0.407322838 & 0.000165754 & 0.006329433 \\
     \cline{2-5}
     & \textit{Jund} &	0.39571235 & \num{5.48104E-11} & \num{2.25303E-08}	 \\
     \cline{2-5}
     & \textit{Gm11808} &	0.391821267 & \num{3.44957E-08} & \num{7.08988E-06}	 \\
     \cline{2-5}
     & \textit{Gadd45g} &	0.381939162 & 0.000398445 & 0.012713849 \\
     \cline{2-5}
     & \textit{Tsc22d3} &0.355528116 & \num{1.98056E-05} & 0.001182921 \\
     \bottomrule
  \end{tabularx}
  \vspace{0.1cm}
   \RaggedRight{\footnotesize \\
    \textbf{\textit{Avg. log\textsubscript{2} FC}}, log fold-change of the average expression of the gene across the sub-populations\\ 
   \textbf{\textit{Adj. p-value}}, adjusted p-value based on Bonferroni correction using all genes in the dataset} 
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[\glslink{de}{DE} genes for Macs-3 sub-population from \glslink{scr}{scRNA-seq}]{\gls{de} genes for Macs-3 noted in \textbf{\autoref{fig:chp2_scrna_macrophages_macs3_dge}}}
     \label{tab:app_scrna_macs3_dge}
  \begin{tabularx}{\textwidth}{xxCCC}
    \toprule
    \rowcolor{headerblue}
    \textbf{Comparison} & \textbf{Gene} & \textbf{Avg. log\textsubscript{2}FC} & \textbf{p-value} & \textbf{Adj. p-value} \\
    \midrule
    \multirow{8}{*}{\makecell{W1\\WD vs Chow}} & \textit{Cxcl2} & 0.576923108 &  \num{2.69795e-05}  & \num{0.001309429} \\
    \cline{2-5}
     & \textit{Il1b} & 0.51836841 & \num{2.59059E-05} & 0.001278956 \\
     \cline{2-5}
     & \textit{Socs3} & 0.507999007 & \num{1.29825E-05} &	0.000760284 \\
     \cline{2-5}
     & \textit{Cxcl1} & 0.488877043 &	0.002187205 &	0.036553155 \\
     \cline{2-5}
     & \textit{Ptgs2} & 0.354411638 &	0.000508486 &	0.012687073 \\
     \cline{2-5}
     & \textit{Fos} & 0.326020259 &	\num{5.53924E-05} &	0.002295384 \\
     \cline{2-5}
     & \textit{Junb} & 0.315533127 &	0.0003139 &	0.009012409 \\
     \cline{2-5}
     & \textit{Mapk14} & 0.288908898 & 0.002053499 &	0.035147575 \\
     
     \midrule

     \multirow{6}{*}{\makecell{W12\\WD vs Chow}} & \textit{Fosb} & 0.555864661 &	\num{3.5604E-08} & \num{1.25116E-05}	 \\
    \cline{2-5}
     & \textit{Mapk14} &	0.394795484 & \num{1.2476E-05} & 0.001461394 \\
     \cline{2-5}
     & \textit{Cxcl2} &	0.365742139 &	0.001563918 &	0.036236549 \\
     \cline{2-5}
     & \textit{Cxcl10} &	0.301159634 &	0.000357731 &	0.015030528 \\
     \cline{2-5}
     & \textit{Trim25} &	0.291879697 &	0.000353963 &	0.015030528 \\
     \cline{2-5}
     & \textit{Ptgs2} &	0.265138665 &	0.000450386 &	0.017323059 \\
     \midrule

      \multirow{7}{*}{\makecell{Chow\\Aging vs Non-aging}} & \textit{Cxcl9} &	1.343096113 &	\num{5.95955E-05} &	0.010682124 \\
    \cline{2-5}
     & \textit{Stat1} &	0.776330394 &	0.000201511 &	0.022297027 \\
     \cline{2-5}
     & \textit{Pml} &	0.534385578 &	0.000595549 &	0.044176545 \\
     \cline{2-5}
     & \textit{Psmb9} &	0.522509255 &	\num{3.26985E-05} &	0.006675035 \\
     \cline{2-5}
     & \textit{Aif1} &	0.52027586 &	\num{5.08612E-08} & \num{4.67224E-05}	 \\
     \cline{2-5}
     & \textit{Birc2} &	0.292074283 &	0.000626371 &	0.045331712 \\
     \cline{2-5}
     & \textit{B2m} &	0.263750171 &	\num{2.04663E-05} &	0.00549014 \\
     \bottomrule
  \end{tabularx}
  \vspace{0.1cm}
   \RaggedRight{\footnotesize \\
    \textbf{\textit{Avg. log\textsubscript{2} FC}}, log fold-change of the average expression of the gene across the sub-populations\\ 
   \textbf{\textit{Adj. p-value}}, adjusted p-value based on Bonferroni correction using all genes in the dataset} 
\end{table}

\clearpage

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
 \caption[Marker genes for islet $\beta$-cells from \glslink{scr}{scRNA-seq}]{Marker genes for islet $\beta$-cells highlighted in \textbf{\autoref{fig:app_scrna_betacells1} A}}
     \label{tab:app_scrna_beta_markers}
  \begin{tabularx}{\textwidth}{xxCCC}
    \toprule
    \rowcolor{headerblue}
    \textbf{Cluster} & \textbf{Gene} & \textbf{Avg. log\textsubscript{2}FC} & \textbf{p-value} & \textbf{Adj. p-value} \\
    \midrule
    \multirow{3}{*}{\makecell{$\beta$-1}} & \textit{Mt1} & 1.473339795 &  0  & 0 \\
    \cline{2-5}
     & \textit{Nupr1} & 1.087103706 & 0 & 0 \\
     \cline{2-5}
     & \textit{G6pc2} & 0.429578306 & 0 & 0 \\
     \cline{2-5}
     & \textit{Glp1r} & 0.345648863 & 0 & 0 \\
     \cline{2-5}
     & \textit{Slc30a8} & 0.335342126 & 0 & 0 \\
     \midrule

     \multirow{4}{*}{\makecell{$\beta$-2}} & \textit{Ndufs2} & 0.611045507 & 0 & 0 \\
    \cline{2-5}
     & \textit{Cox6a2} & 0.531399941 & 0 & 0 \\
     \cline{2-5}
     & \textit{Ero1lb} & 0.40965907 & 0 & 0 \\
     \midrule

      \multirow{5}{*}{\makecell{$\beta$-3}} & \textit{Cd81} & 1.40144447 & 0 & 0 \\
    \cline{2-5}
     & \textit{Folr1} & 1.010963942 & 0 & 0 \\
     \cline{2-5}
     & \textit{Gnas} & 0.995773308 & 0 & 0 \\
     \cline{2-5}
     & \textit{Fxyd3} & 0.836848657 & 0 & 0 \\
     \bottomrule
  \end{tabularx}
  \vspace{0.1cm}
   \RaggedRight{\footnotesize \\
    \textbf{\textit{Avg. log\textsubscript{2} FC}}, log fold-change of the average expression of the gene across the sub-populations\\ 
   \textbf{\textit{Adj. p-value}}, adjusted p-value based on Bonferroni correction using all genes in the dataset} 
\end{table}

\end{comment}


%\begin{comment}

\section{Additional results Chapter 3}

\begin{table}[h]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Summary of the seven studies}
  \label{tab:app_chp3_study}

  \begin{tabularx}{\textwidth}{W{2.5cm} X W{1cm} W{1.7cm} W{1.5cm} W{1.3cm}}
    \toprule
    \rowcolor{headerblue}
    \textbf{Name} & \textbf{Description} & \textbf{\textit{N}} & \textbf{Age} & \textbf{Sex} & \textbf{Source} \\
    \midrule
    \RaggedRight{Maturation/Aging} &  & 2 & 3 wks \& 2 years & Male & In-house \\
    \hline
    %\rowcolor{rowblue}
    Meal feeding &  & 2 & 12-14 wks & \RaggedRight{Male \& Female} & In-house \\
    \hline
    Diet-induced obesity &  & 4 & 10 wks \& 21 wks & Male & In-house \\
    \hline
    %\rowcolor{rowblue}
    Mild genetic obesity (\textit{db/db}) &  & 4 & 6 wks \& 9 wks & Male & In-house \\
    \hline
    Severe genetic obesity (\textit{ob/ob}) &  & 2 & ??? & Male & External \textbf{\cite{chung_endocrine-exocrine_2020}} \\
    \hline
    %\rowcolor{headerblue}
    Insulin receptor blockade (S961) &  & 2 & ??? & Male & In-house \\
    \hline
    \RaggedRight{Partial β-cell ablation (STZ)} &  & 2 & 8 wks & Male & External \textbf{\cite{sachs_targeted_2020}} \\
    \bottomrule
   
  \end{tabularx}
\end{table}
\textit{N}, Number of Groups

\clearpage

\begin{table}[h]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Total number of all cells and $\beta$-cells across all groups within the seven studies}
  \label{tab:app_chp3_cellnumbers}

  \begin{tabularx}{\textwidth}{W{2.5cm}  W{2.5cm}  C{4cm}  C{4cm}}
    \toprule
    \rowcolor{headerblue}
    \thead{Study} & \thead{Samples} & \thead{\textit{N cells}\textsuperscript{*}\\(Total)} & \thead{\textit{N cells}\textsuperscript{**}\\(β-cells)$\dagger$} \\
    \midrule
    \multirow{2}{*}{\makecell{Maturation/\\Aging}} & 3-wks old & 6361  & 968 (871)  \\
     \cline{2-4}
     & 2 years old & 14602 & 6743 (6739) \\
     \midrule

    \multirow{2}{}{\makecell{Meal Feeding}} & Fast & 1527  & 419 (419)  \\
    \cline{2-4}
    & Fed & 1229 & 331 (331) \\
    \midrule

    \multirow{4}{*}{\makecell{Diet-induced\\obesity}} & Chow 1wks & 10584  & 2529 (2519)  \\
    \cline{2-4}
    & Chow 12wks & 9154 & 2971 (2970) \\
    \cline{2-4}
    & WD 1wks & 12193 & 2858 (2851) \\
    \cline{2-4}
    & WD 12wks & 16115 & 3768 (3724) \\
    \midrule
    \multirow{4}{*}{\makecell{Mild genetic\\obesity}} & \textit{dbHet} 6wks & 1777  & 732 (726)  \\
    \cline{2-4}
    & \textit{dbHet} 9wks & 4779 & 1608 (1583) \\
    \cline{2-4}
    & \textit{db/db} 6wks & 6546 & 3777 (3674) \\
    \cline{2-4}
    & \textit{db/db} 9wks & 3644 & 2224 (2125) \\
    \midrule
    \multirow{2}{*}{\makecell{Severe genetic\\obesity}} & Wild type & 7827  & 6412 (6375)  \\
    \cline{2-4}
    & \textit{ob/ob} & 8812 & 8280 (8184) \\
    \midrule
    \multirow{2}{*}{\makecell{Insulin receptor\\blockade}} & Vehicle & 890  & 491 (491)  \\
    \cline{2-4}
    & S961 & 1022 & 779 (719) \\
    \midrule
    \multirow{2}{*}{\makecell{Partial β-cell\\ablation}} & Vehicle & 8338  & 5816 (5790)  \\
    \cline{2-4}
    & \gls{stz} & 8791 & 1324 (1307) \\
    \bottomrule
   \end{tabularx}
   \vspace{0.1cm}
   \RaggedRight{\footnotesize \\ \textsuperscript{*}After initial preprocessing and quality control\\ \textsuperscript{**}After additional quality control\\ $\dagger$ The number of non-proliferating β-cells are denoted in parentheses.\\\textbf{WD}, Western Diet; \textbf{STZ}, streptozotocin}
\end{table}



\begin{table}[htbp]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Regulon Modules}
  \label{tab:app_chp3_regulons}

  \begin{tabularx}{\textwidth}{W{2.5cm} C}
    \toprule
    \rowcolor{headerblue}
    \textbf{Module} & \textbf{Regulons} \\
    \midrule
    %\rowcolor{mod1green}
    Module-1 & \cellcolor[cmyk]{0.45,0,0.7,0} \RaggedRight{\textit{\textbf{Atf5(+)},
    Bach1(+),
    Bhlhe41(+),
    Creb3l1(+),
    Dbp(+),
    Egr1(+),
    Elk3(+),
    Fosb(+),
    Foxp1(+),
    Hlf(+),
    Hmgb1(+),
    Hoxb3(+),
    \textbf{Isl1(+)},
    \textbf{Kdm2b(+)},
    \textbf{Klf5(+)},
    \textbf{Mafa(+)},
    Meox1(+),
    Mlxipl(+),
    Mnx1(+),
    \textbf{Neurod1(+)},
    Nfix(+),
    \textbf{Nkx6-1(+)},
    Pbx1(+),
    \textbf{Pdx1(+)},
    Pknox1(+),
    Pou2f1(+),
    Six4(+),
    Sox4(+),
    Tcf4(+),
    \textbf{Tcf7l2(+)},
    Tef(+),
    \textbf{Vdr(+)},
    Zfhx3(+)}} \\
    %\hline
    \midrule
    Module-2 & \cellcolor[cmyk]{0.22,0.51,0,0} \RaggedRight{\textit{\textbf{Arx(+)},
    Ascl1(+),
    Barx2(+),
    Bhlhe22(+),
    Brca1(+),
    Egr2(+),
    Ehf(+),
    \textbf{Fos(+)},
    Foxp2(+),
    Gata6(+),
    Hmgb2(+),
    \textbf{Jun(+)},
    Jund(+),
    Klf6(+),
    Mafb(+),
    Mef2c(+),
    Nr1d1(+),
    Onecut2(+),
    Rorc(+),
    Shox2(+),
    Taf7(+),
    Thrb(+),
    Trps1(+)}} \\
\midrule
    Module-3 & \cellcolor[cmyk]{0,0.52,0.56,0.03} \RaggedRight{\textit{Arid5a(+),
Atf3(+),
Bcl3(+),
Bhlhe40(+),
Cebpb(+),
Cebpd(+),
Ddit3(+),
Egr3(+),
Egr4(+),
Elf3(+),
Irf1(+),
Irf7(+),
Irf8(+),
Irf9(+),
Junb(+),
Klf4(+),
Lhx1(+),
Maff(+),
Nfia(+),
Nfil3(+),
Nfkb2(+),
Pbx4(+),
Relb(+),
Stat1(+),
Stat2(+),
Stat3(+),
Tagln2(+),
Zfp467(+)}} \\
\midrule
    Module-4 & \cellcolor[cmyk]{0,0.89,0.67,0.1} \RaggedRight{\textit{Bach2(+),
Bhlha15(+),
Crem(+),
E2f1(+),
Etv5(+),
Gtf2b(+),
Hoxb2(+),
Mybl1(+),
Nr1h3(+)}} \\
\midrule
    Module-5 & \cellcolor[cmyk]{0,0.35,1,0.14} \RaggedRight{\textit{Barhl1(+),
Bcl11a(+),
Cdx4(+),
Chd2(+),
Creb5(+),
Dab2(+),
E2f2(+),
E2f8(+),
Ebf1(+),
Epas1(+),
Ets1(+),
Ezh2(+),
Foxs1(+),
Hes1(+),
Hmgb3(+),
Hoxb4(+),
Irx1(+),
Myc(+),
Nfe2l2(+),
Nfib(+),
Nkx6-2(+),
Nkx6-3(+),
Nr1i2(+),
Pou3f1(+),
Pou3f4(+),
Pou6f2(+),
Rnaseh2c(+),
Sox18(+),
Sox9(+),
Tbx1(+),
Tff3(+)}} \\
    
    \bottomrule
   
  \end{tabularx}

\end{table}





%\end{comment}