%!TEX root = ../thesis.tex
% ******************************* Thesis Appendix A ****************************
\chapter{Supplementary Tables} 
\clearpage

\section{Additional results Chapter 2}

\begin{table}[ht!]
  \renewcommand{\arraystretch}{1.2} % More space between rows
  \centering
  \begin{tabularx}{\textwidth}{|G|C|C|}
    \hline
    \rowcolor{headerblue}
    \textbf{Isotopes} & \textbf{Marker} & \textbf{Cell Type} \\
    \hline
    141Pr & Insulin & $\beta$-cells \\
    \hline
    %\rowcolor{rowgray}
    142Nd & B220 & B-Lymphocytes \\
    \hline
    143Nd & CD3 & T-Lymphocytes \\
    \hline
    %\rowcolor{rowgray}
    144Nd & CD138 (\textit{Sdc1}) & Plasma cells \\
    \hline
    %\rowcolor{selectedrow}
    145Nd & CD8 & CD8\textsuperscript{+} T-Lymphocytes \\
    \hline
    146Nd & CD31 (\textit{Pecam1}) & Endothelial Cells \\
    \hline
    147Sm & CD16 (\textit{Fcgr3}) & Monocytes \\
    \hline
    149Sm & CD19 (\textit{Cd19}) & B-Lymphocytes \\
    \hline
    150Nd & CD11b &  \\
    \hline
    151Eu & CD44 (\textit{Cd44}) &  \\
    \hline
    153Eu & CD127 (\textit{Il7r}) &  \\
    \hline
    154Sm & CD69 (\textit{Cd69}) &  \\
    \hline
    158Gd & CRTH2 (\textit{Ptgdr2}) & Innate Lymphoid Cells (ILCs) \\
    \hline
  %\rowcolor{rowgray}
    159Tb & Eomes & CD4\textsuperscript{+} T-Lymphocytes with cytotoxic activity \\
    \hline
    %\rowcolor{selectedrow}
    160Gd & Ly6G (\textit{Ly6g}) & Granulocytes \\
    \hline
  \end{tabularx}
  \caption[]{}
  \label{tab2-1}
\end{table}

\clearpage


\section{Additional results Chapter 3}

\begin{table}[h]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Summary of datasets used in this study and their corresponding metadata}
  \label{tab3-1}

  \begin{tabularx}{\textwidth}{W{2.5cm} X W{1cm} W{1.7cm} W{1.5cm} W{1.3cm}}
    \toprule
    %\rowcolor{headerblue}
    \textbf{Name} & \textbf{Description} & \textbf{\textit{N}} & \textbf{Age} & \textbf{Sex} & \textbf{Source} \\
    \midrule
    \RaggedRight{Maturation/Aging} &  & 2 & 3 wks \& 2 years & Male & In-house \\
    \hline
    %\rowcolor{rowblue}
    Meal feeding &  & 2 & 12-14 wks & \RaggedRight{Male \& Female} & In-house \\
    \hline
    Diet-induced obesity &  & 4 & 10 wks \& 21 wks & Male & In-house \\
    \hline
    %\rowcolor{rowblue}
    Mild genetic obesity (\textit{db/db}) &  & 4 & 6 wks \& 9 wks & Male & In-house \\
    \hline
    Severe genetic obesity (\textit{ob/ob}) &  & 2 & ??? & Male & External \textbf{\cite{chung_endocrine-exocrine_2020}} \\
    \hline
    %\rowcolor{headerblue}
    Insulin receptor blockade (S961) &  & 2 & ??? & Male & In-house \\
    \hline
    \RaggedRight{Partial β-cell ablation (STZ)} &  & 2 & 8 wks & Male & External \textbf{\cite{sachs_targeted_2020}} \\
    \bottomrule
   
  \end{tabularx}
\end{table}
\textit{N}, Number of Groups

\begin{table}[h]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Number of total cells and }
  \label{tab:3-3}

  \begin{tabularx}{\textwidth}{W{2.5cm}  W{2.5cm}  C{4cm}  C{4cm}}
    \toprule
    %\rowcolor{headerblue}
    \thead{Study} & \thead{Samples} & \thead{\textit{N cells}\textsuperscript{*}\\(Total)} & \thead{\textit{N cells}\textsuperscript{**}\\(β-cells)$\dagger$} \\
    \midrule
    \multirow{2}{*}{\makecell{Maturation/\\Aging}} & 3-wks old & 6361  & 968 (871)  \\
     \cline{2-4}
     & 2 years old & 14602 & 6743 (6739) \\
     \midrule

    \multirow{2}{}{\makecell{Meal Feeding}} & Fast & 1527  & 419 (419)  \\
    \cline{2-4}
    & Fed & 1229 & 331 (331) \\
    \midrule

    \multirow{4}{*}{\makecell{Diet-induced\\obesity}} & Chow 1wks & 10584  & 2529 (2519)  \\
    \cline{2-4}
    & Chow 12wks & 9154 & 2971 (2970) \\
    \cline{2-4}
    & WD 1wks & 12193 & 2858 (2851) \\
    \cline{2-4}
    & WD 12wks & 16115 & 3768 (3724) \\
    \midrule
    \multirow{4}{*}{\makecell{Mild genetic\\obesity}} & \textit{dbHet} 6wks & 1777  & 732 (726)  \\
    \cline{2-4}
    & \textit{dbHet} 9wks & 4779 & 1608 (1583) \\
    \cline{2-4}
    & \textit{db/db} 6wks & 6546 & 3777 (3674) \\
    \cline{2-4}
    & \textit{db/db} 9wks & 3644 & 2224 (2125) \\
    \midrule
    \multirow{2}{*}{\makecell{Severe genetic\\obesity}} & Wild type & 7827  & 6412 (6375)  \\
    \cline{2-4}
    & \textit{ob/ob} & 8812 & 8280 (8184) \\
    \midrule
    \multirow{2}{*}{\makecell{Insulin receptor\\blockade}} & Vehicle & 890  & 491 (491)  \\
    \cline{2-4}
    & S961 & 1022 & 779 (719) \\
    \midrule
    \multirow{2}{*}{\makecell{Partial β-cell\\ablation}} & Vehicle & 8338  & 5816 (5790)  \\
    \cline{2-4}
    & \gls{stz} & 8791 & 1324 (1307) \\
    \bottomrule
   \end{tabularx}
   \vspace{0.1cm}
   \RaggedRight{\footnotesize \\ \textsuperscript{*}After initial preprocessing and quality control\\ \textsuperscript{**}After additional quality control\\ $\dagger$ The number of non-proliferating β-cells are denoted in parentheses.\\\textbf{WD}, Western Diet; \textbf{STZ}, streptozotocin}
\end{table}



\begin{table}[htbp]
\renewcommand{\arraystretch}{2}
  \centering
  \caption{Regulon Modules}
  \label{tab3-2}

  \begin{tabularx}{\textwidth}{W{2.5cm} C}
    \toprule
    \textbf{Module} & \textbf{Regulons} \\
    \midrule
    %\rowcolor{mod1green}
    Module-1 & \cellcolor[cmyk]{0.45,0,0.7,0} \RaggedRight{\textit{\textbf{Atf5(+)},
    Bach1(+),
    Bhlhe41(+),
    Creb3l1(+),
    Dbp(+),
    Egr1(+),
    Elk3(+),
    Fosb(+),
    Foxp1(+),
    Hlf(+),
    Hmgb1(+),
    Hoxb3(+),
    \textbf{Isl1(+)},
    \textbf{Kdm2b(+)},
    \textbf{Klf5(+)},
    \textbf{Mafa(+)},
    Meox1(+),
    Mlxipl(+),
    Mnx1(+),
    \textbf{Neurod1(+)},
    Nfix(+),
    \textbf{Nkx6-1(+)},
    Pbx1(+),
    \textbf{Pdx1(+)},
    Pknox1(+),
    Pou2f1(+),
    Six4(+),
    Sox4(+),
    Tcf4(+),
    \textbf{Tcf7l2(+)},
    Tef(+),
    \textbf{Vdr(+)},
    Zfhx3(+)}} \\
    %\hline
    \midrule
    Module-2 & \cellcolor[cmyk]{0.22,0.51,0,0} \RaggedRight{\textit{\textbf{Arx(+)},
    Ascl1(+),
    Barx2(+),
    Bhlhe22(+),
    Brca1(+),
    Egr2(+),
    Ehf(+),
    \textbf{Fos(+)},
    Foxp2(+),
    Gata6(+),
    Hmgb2(+),
    \textbf{Jun(+)},
    Jund(+),
    Klf6(+),
    Mafb(+),
    Mef2c(+),
    Nr1d1(+),
    Onecut2(+),
    Rorc(+),
    Shox2(+),
    Taf7(+),
    Thrb(+),
    Trps1(+)}} \\
\midrule
    Module-3 & \cellcolor[cmyk]{0,0.52,0.56,0.03} \RaggedRight{\textit{Arid5a(+),
Atf3(+),
Bcl3(+),
Bhlhe40(+),
Cebpb(+),
Cebpd(+),
Ddit3(+),
Egr3(+),
Egr4(+),
Elf3(+),
Irf1(+),
Irf7(+),
Irf8(+),
Irf9(+),
Junb(+),
Klf4(+),
Lhx1(+),
Maff(+),
Nfia(+),
Nfil3(+),
Nfkb2(+),
Pbx4(+),
Relb(+),
Stat1(+),
Stat2(+),
Stat3(+),
Tagln2(+),
Zfp467(+)}} \\
\midrule
    Module-4 & \cellcolor[cmyk]{0,0.89,0.67,0.1} \RaggedRight{\textit{Bach2(+),
Bhlha15(+),
Crem(+),
E2f1(+),
Etv5(+),
Gtf2b(+),
Hoxb2(+),
Mybl1(+),
Nr1h3(+)}} \\
\midrule
    Module-5 & \cellcolor[cmyk]{0,0.35,1,0.14} \RaggedRight{\textit{Barhl1(+),
Bcl11a(+),
Cdx4(+),
Chd2(+),
Creb5(+),
Dab2(+),
E2f2(+),
E2f8(+),
Ebf1(+),
Epas1(+),
Ets1(+),
Ezh2(+),
Foxs1(+),
Hes1(+),
Hmgb3(+),
Hoxb4(+),
Irx1(+),
Myc(+),
Nfe2l2(+),
Nfib(+),
Nkx6-2(+),
Nkx6-3(+),
Nr1i2(+),
Pou3f1(+),
Pou3f4(+),
Pou6f2(+),
Rnaseh2c(+),
Sox18(+),
Sox9(+),
Tbx1(+),
Tff3(+)}} \\
    
    \bottomrule
   
  \end{tabularx}

\end{table}
%\textit{N}, Number of Samples


% \begin{table}[h]
%     \centering
%     \begin{tabular}{c|c c c c c c}
%     &          0 & 5 & 10 & 15 & 20 & 25  \\
%     \hline
%     PCA      & 41 & 77 & 78 & 88 & 86 & 84 \\
%     PEER     & 41 & 70 & 70 & 71 & 63 & 87 \\
%     MOFA     & 41 & 64 & 76 & 75 & 77 & 73 \\
%     MOFA-ns  & 41 & 74 & 78 & 73 & 74 & 72 \\
%     LDVAE    & 41 & 58 & 54 & 71 & 74 & 73 \\
%     \end{tabular}
%     \caption[Covariate comparison in terms of replication of matched bulk results]{\textbf{Number of m-bulk-replicated eQTL across covariates.} \\
%     Equivalent to \textbf{Table \ref{tab:covariates_replication}} (i.e. mean as aggregation method, 1,421 chromosome 2 genes tested only, FDR<10\% and same direction of effect as replication), except considering replication using bulk with matched samples only (m-bulk; n=88).}
%     \label{tab:covariates_replication_matched_bulk}
% \end{table}

% \newpage

% \section{Additional results for Chapter 4}

% \begin{table}[h]
%     \centering
%     \begin{tabular}{c | c c c c c}
%      & eGenes & genes tested & cells & unique donors & unique samples*  \\
%     \hline
%     iPSC (day0) & 1,833 & 10,840 & 9,661 & 111 & 136 \\
%     mesendo & 1,702 & 10,924 & 9,809 & 123 & 224 \\
%     defendo & 1,342 & 10,901 & 10,924 & 116 & 238\\
%     transitioning & 227 & 10,924 & 6,387 & 118 & 313 \\
%     day1 & 1,181 & 10,787 & 9,443 & 111 & 138 \\
%     day2 & 718 & 10,788 & 8,455 & 105 & 116 \\
%     day3 & 631 & 10,765 & 8,485 & 108 & 127 \\
%     \end{tabular}
%     \caption[Summary of the type and number of eQTL]{\textbf{Summary of the type and number of eQTL}.\\
%     Including all eQTL discovered based on single cell RNA traits, both at the level of our computationally-defined stages (iPSC, mesendo and defendo) and the time points of collection by design (day1, day2 and day3).
%     We note that in the stage definition a number of cells ($\sim$20\%) were excluded and considered to not belong to any well defined stage (see \textbf{page \pageref{fig:endodiff_stages}}), exclusively for the purposes of stage-level eQTL mapping.
%     We included here an eQTL map of such transitioning population, which as expected was a very underpowered analysis.
%     Additionally, we added results for day2 cells, for completeness.
%     Shown are the number of genes that were considered for eQTL mapping, as well as the number of genes for which a eQTL was detected.
%     *Samples are donor-experiment(-time point) combinations which were effectively used for testing (see \textbf{section \ref{sec:endodiff_eqtl}}).}
%     \label{tab:endodiff_eqtl_summary}
% \end{table}
% % number of eqtl, sample size etc?

% \vspace{10mm}

% \begin{table}[h]
%     \centering
%     \begin{tabular}{c c c}
%      Antibody raised against & Catalog number & Company  \\
%     \hline
%     Histone H3 & ab1791 & Abcam \\
%     Histone H3 (tri methyl K4) & ab8580 & Abcam \\
%     Histone H3 (tri methyl K27) & C15200181 (MAb-181-050)
%  & Diagenode\\
%     Histone H3 (mono methyl K4) & ab8895 & Abcam \\
%     Histone H3 (acetyl K27) & ab4729 & Abcam \\
%     Histone H3 (tri methyl K36) & ab9050 & Abcam \\
%     \end{tabular}
%     \caption[Antibodies used for ChIP-seq experiments]{\textbf{Antibodies used for the ChIP-seq experiments}.\\
%     Experimental methods for this analysis are described in \textbf{section \ref{sec:endodiff_chipseq}}, and these data were used for analysis shown in \textbf{Fig. \ref{fig:endodiff_stage_specific_eqtl}} and \textbf{\ref{fig:endodiff_dynamic_eqtl_enrichment}}.} 
%     \label{tab:endodiff_chipseq_antibodies}
% \end{table}

% \clearpage

% \section{Additional information for Chapter 5}

% \begin{table}[h]
%     \centering
%     \begin{tabular}{c c c c c}
%     trait & category & study & n & n (replication)  \\
%     \hline
%     Alzheimer's Disease (late onset) &  neurodegenerative & \cite{lambert2013meta}  & 55,134 & 19,884\\
%     Parkinson's Disease  &  neurodegenerative & \cite{nalls2019expanding} & 442,271 & -\\
%     Schizophrenia   &  neurodevelopmental & \cite{schizophrenia2014biological} & 150,064 & -\\
%     Bipolar disorder & neurodevelopmental & \cite{hou2016genome} & 34,950 & 5,305\\
%     Neuroticism & personality & \cite{okbay2016genetic} & 170,911 & -\\
%     Neuroticism & personality & \cite{luciano2018association} & 329,821 & 122,867\\
%     Neuroticism & personality & \cite{turley2018multi} & 168,105 & -\\
%     Depression (broad) & behavioural & \cite{howard2018genome} & 322,580 & -\\
%     Educational attainment & intelligence & \cite{davies2016genome} & 111,114 & - \\
%     Paternal history of Alzheimer's disease &  neurodegenerative & \cite{marioni2018gwas} & 260,279 & - \\
%     Family history of Alzheimer's disease &  neurodegenerative & \cite{marioni2018gwas} & 314,278 & -\\
%     Maternal history of Alzheimer's disease &  neurodegenerative & \cite{marioni2018gwas} & 288,676 & -\\
%     Depressed affect & behavioural & \cite{nagel2018meta} & 357,957 & -\\
%     Cognitive performance & intelligence & \cite{lee2018gene} & 257,841 & -\\
%     Sleeplessness / insomnia & personality & \cite{neale2018gwas} & 360,738 & -\\
%     Nervous feelings & behavioural & \cite{neale2018gwas} & 351,829 & -\\
%     Worrier / anxious feelings & behavioural & \cite{neale2018gwas} & 351,833 & - \\
%     Tense / 'highly strung' & behavioural & \cite{neale2018gwas} & 350,159 & -\\
%     Suffer from 'nerves' & behavioural & \cite{neale2018gwas} & 348,082 & -\\
%     Neuroticism score & personality & \cite{neale2018gwas} & 293,006 & - \\
%     Intelligence questions\footnotemark & intelligence & \cite{neale2018gwas} & 117,131 & -\\
%     Risk taking & behavioural & \cite{neale2018gwas} & 348,549 & -\\
%     College or University degree & intelligence & \cite{neale2018gwas} & 357,549 & - \\
%     A levels/AS levels or equivalent & intelligence & \cite{neale2018gwas} & 357,549 & -\\
%     Other professional qualifications  & intelligence & \cite{neale2018gwas} & 357,549 & -\\
%     \end{tabular}
%     \caption[Neurological traits used for the colocalisation analysis]{\textbf{Neurological traits used for the colocalisation analysis}.\\
%     Table compiled by Natsuhiko Kumasaka.
%     Traits used for the colocalisation analysis in \textbf{Chapter 5} (\textbf{section \ref{sec:neuroseq_coloc}}).
%     The corresponding publication and sample size (both for the initial study and, when available for a replication study) are indicated.
%     Additionally, traits are divided into broad categories.}
%     \label{tab:coloc_neuro_traits}
% \end{table}

% \footnotetext{Number of fluid intelligence questions attempted within time limit.}