%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Fifth Chapter *********************************
%*******************************************************************************

\chapter{Methods}
\label{chp:methods}

\newpage\null\thispagestyle{empty}\newpage

\section{Computational methods for \autoref{chp:diet_aging}}
\label{sec:chp2_methods}

\subsection[\glslink{scr}{scRNA-seq} data generation]{\large \gls{scr} data generation}
\vspace{3mm}
\begin{Comment2}
\vspace{1mm}
\hspace{-3mm}
The following section is based on text originally provided by Dr. Han Zhu, and has been revised by me for this thesis.
\vspace{1mm}
\end{Comment2}
\vspace{3mm}

In this study, 10-week-old and 78-week-old male C57BL/6J mice were procured from The Jackson Laboratory and housed in vivaria following approved standards and procedures. The animals were acclimatized for one week and maintained under standard conditions with a 12-hour light/dark cycle and ad libitum access to water and standard rodent chow. The 78 weeks old mice continued under these conditions for additional 27 weeks to meet the aging criteria. To induce overnutrition, the standard chow diet was substituted with a \acrfull{wd} comprising 42\% of calories from fat and 42.7\% from carbohydrates. Islets were isolated following a modified standard protocol involving perfusion of Collagenase P through the hepatic bile duct and mechanical dissociation. Islets were meticulously purified using density gradient separation and handpicking to minimize acinar cell contamination. For single-cell analysis, isolated islets underwent enzymatic dissociation into a single-cell suspension, and cells expressing CD45 were sorted via \acrfull{facs}. Sorted cells were processed using the 10X Chromium Controller for \gls{gem} formation and barcoding, followed by \glslink{cdna}{cDNA} synthesis and library construction with 10X Next \gls{gem} reagents. The libraries were sequenced using Illumina platforms, ensuring high-quality data for downstream analysis. 

\subsection[\glslink{imc}{IMC} data generation and processing]{\large \gls{imc} data generation and processing}
\vspace{3mm}
\begin{Comment2}
\vspace{1mm}
\hspace{-3mm}
The following section is based on text originally provided by Dr. Matthias Barone, and has been revised by me for this thesis.
\vspace{1mm}
\end{Comment2}
\vspace{3mm}
For \acrfull{ihc}, tissue sections were prepared by de-paraffinizing in xylene, rehydrating in graded ethanol, and treating with antigen retrieval buffer. The sections were blocked, stained with primary and secondary antibodies, and imaged using a microscope ensuring constant exposure times between acquisitions. For \gls{imc}, sections were similarly prepared but stained with metal-conjugated antibodies, using Iridium (191Ir and 193Ir) for \glslink{dna}{DNA} staining. The selected \glsreset{roi} \glspl{roi} in the stained tissue sections were then imaged pixel-by-pixel by the Hyperion™ Tissue Imager and analyzed by time-of-flight mass spectrometry (\glslink{cytof}{CyTOF}) with the Helios™ Mass Cytometer (Fluidigm).\\
\par In data processing, \glspl{roi} were segmented using the Bodenmiller pipeline with modifications \textbf{\cite{eling_imc_2022}}, including hot pixel filtering and cell-wise segmentation via python scripts and CellProfiler \textbf{\cite{carpenter_cellprofiler_2006,mcquin_cellprofiler_2018}}. This was followed by pixel-based training in Ilastik \textbf{\cite{berg_ilastik_2019}} for accurate cellular object identification. Euclidean distance based islet and lymph node masks were created with a macro written for ImageJ \textbf{(\cite{schneider_nih_2012})}. Subsequently, data were processed in R using adapted SPECTRE \textbf{\cite{ashhurst_integration_2022}} code for spillover correction, batch correction, signal normalization, cellular gating, and data transformation.\\
\par For data analysis, the marker signals were normalized using z-scores and clustered using Phenograph via \textit{cytofkit} package (v0.99.0) and visualized with \textit{ComplexHeatmap} (v2.12.0). In order to visualize single-cells, dimensionality reduction was performed with \gls{umap} algorithm with \textit{uwot} package (v0.1.14). After individual clustering of each subset, euclidean distance maps were loaded as matrices into R and the spatial position of each cell (center of mass) was mapped onto the supplied maps and cells were flagged accordingly.


\subsection[\glslink{scr}{scRNA-seq} data processing and analysis]{\gls{scr} data processing and analysis}


\subsubsection{\large Processing of 10x output}
The Cell Ranger analysis pipeline (v4.0.0), provided by 10x Genomics, was used for: \textbf{(i)} aligning the reads from the de-multiplexed FASTQ files to the mm10 (2020-A) genome build (also from 10x Genomics); \textbf{(ii)} filtering and \textbf{(iii)} barcode and \gls{umi} counting and \textbf{(iv)} generating gene expression matrices. The cell-calling algorithm in Cell Ranger automatically excludes barcodes that correspond to background noise, yielding a filtered gene expression matrix, which was used for all downstream analysis. Next, to identify potential technical artifacts of cell doublets, we applied Scrublet (v0.2.1) \textbf{\cite{wolock_scrublet_2019}}, on a per-sample basis (the \textit{expected\textunderscore doublet\textunderscore rate} parameter was manually adjusted depending on the number of cells obtained in a particular sample. 

\subsubsection{\large Data preprocessing}
The filtered feature-barcode matrix (in HDF5 format) of individual samples were imported into R via the \textit{Read10X\textunderscore h5()} function, provided by Seurat v4 \textbf{\cite{hao_integrated_2021}} package. The individual matrices were then converted into Seurat objects with the \textit{CreateSeuratObject()} function, while filtering out: \textbf{(i)} features detected in less than 3 cells (min.cells = 3) and \textbf{(ii)} cells with less than 500 features detected (min.features = 500). The percentage of reads mapping to all mitochondrial genes (\textit{mt-}\textsuperscript{*}), L-ribosomal proteins (\textit{Rpl}\textsuperscript{*}) and S-ribosomal proteins (\textit{Rps}\textsuperscript{*}) were computed with the \textit{PercentageFeatureSet()} function. Cells with over 12.5\% of counts due to mitochondrial genome were excluded. Following this, cells identified as putative doublets by Scrublet were also removed. Some of the doublet artifacts might have been removed in earlier quality control (QC) steps. Then, for each sample, the ambient \glslink{mrna}{mRNA} contamination was removed using SoupX (v---) \textbf{\cite{young_soupx_2020}}, with a fixed contamination fraction of 5\%. This threshold performed better compared to: \textbf{(i)} no ambient \glslink{mrna}{mRNA} removal, \textbf{(ii)} automated removal, and \textbf{(iii)} estimating the contamination fraction using islet hormone and transcription-factor markers (Alpha: \textit{Gcg, Arx, Ttr}; Beta: \textit{Ins1, Ins2, Iapp}; Delta: \textit{Sst} and Gamma: \textit{Ppy, Pyy}). The counts were adjusted and rounded to the nearest integer with \textit{adjustCounts()} function.  

\subsubsection{\large Data normalization and integration}
The QC and SoupX-corrected count matrices were log-normalized using \textit{NormalizeData()} function and 3000 \glspl{hvf} were selected using \textit{FindVariableFeatures()} function. To speed up the integration process, \gls{rpca} was applied, wherein, anchors between any two samples are determined by projecting each sample into others’ PCA space. For this purpose, individual samples were scaled across the \glspl{hvf} using \textit{ScaleData()} function and 50 \glspl{pc} were computed with \textit{RunPCA()} function. Following this, anchors between the samples were identified using \textit{FindIntegrationAnchors()} with parameter reduction = ‘rpca’. The identified anchors were then used for integrating the samples via \textit{IntegrateData()} function. The integrated data was further scaled and centered across the \glspl{hvf}, and subjected to \gls{pc} analysis. The top 30 \glspl{pc} were further used to build a neighborhood graph with \textit{FindNeighbors(…, dims = 1:30, …)} function. The computed graph was used to: \textbf{(i)} identify clusters of cells using the original Louvain algorithm at a resolution parameter of 0.2, using the \textit{FindClusters(…, resolution = 0.2, …)} function, which yielded 19 clusters; \textbf{(ii)} reduce the dimensionality of the integrated data from 30 \glspl{pc} to a two-dimensional projection via the \gls{umap} algorithm, implemented in the \textit{RunUMAP(…, min.dist = 0.1, …)} function.  


\subsubsection{\large Immune subset, Macrophages and T-cells}
To further characterize the dataset, we first annotated the integrated data based on the expression of \textit{Ptprc} (CD45) gene, allowing us to distinguish the ‘Immune’ populations and the `Endocrine-\&-Others' populations. The cells annotated as `Immune' were then subsetted from the full integrated data and cells expressing hormone markers (\textit{Gcg, Ins1, Ins2, Sst} and \textit{Ppy}) were excluded from further analysis. Following this, the immune subset was split into individual batches with the \textit{SplitObject()} function. This was necessary as we excluded batches which were not enriched for the immune populations, from downstream analysis. Following this, we estimated the cell cycle phase (G1, S or G2/M), per batch, based on the expression of G2/M and S phase markers, using the \textit{CellCycleScoring()} function. Next, we removed cells with > 17\% of transcripts mapping to \glspl{dag}, as these could be stressed or dying, as previously reported \textbf{\cite{}} and removed the following HSP genes: \textit{Hspa1a, Hspa1b, Hspb1, Hsph1, Hsp90aa1, Hspa8} and \textit{Hsp90ab1}  from further analysis. The individual batches were then processed and integrated in a similar fashion as outlined in the above section, to obtain an integrated immune-specific subset. The individual cell-types were annotated with canonical markers: Macrophages (\textit{Lyz2, Adgre1}), B-Lymphocytes (\textit{Cd19, Cd79a} and \textit{Cd79b}), Plasma (\textit{Jchain}), T-\&-other-Lymphocytes (\textit{Cd3e}), `Monocytic’ (\textit{Fn1}), Neutrophils (\textit{S100a9, S100a8}) and Dendritic cells – cDC1 (\textit{Xcr1}), cDC2 (\textit{Flt3}) and pDC (\textit{Siglech}).\\\\
In order to generate a macrophage specific object, we subsetted the cells annotated as 'Macrophages' and 'Proliferating-Endocrine' from the full integrated immune dataset. Next, we perfomed additional QC steps in order to remove lymphoid populations. Similar to the re-integration of the full immune data, we split the macropahges into batches and re-integrated as previously described.\\\\ 
In case of T-cells, we subsetted cells annotated as 'T-cells \& others' and 'Proliferating cells' from the full integrated immune data. We excluded cells with expression of markers related to myeloid populations as well as B-cells. The remaining cells were re-integrated in a batch-wise manner similar to the full immune dataset and the macrophage specific subset.

\subsubsection{\large Endocrine subset, polyhormonal annotation and $\beta$-cell re-integration}
The cells annotated as `Endocrine-\&-Others' were subsetted from the full integrated data. Next, the subset was split into individual batches with the \textit{SplitObject()} function, and immune-specific batches were excluded from further analysis. Similar to the immune subset, we estimated the cell-cycle phase, per batch, with the \textit{CellCycleScoring()} function and computed the percentage of expression of \glspl{dag} using the  \textit{PercentageFeatureSet()} function. The endocrine batches were normalized and integrated as described in the above section. The major endocrine sub-populations were annotated based on the expression of hormone genes (Alpha ($\alpha$) – \textit{Gcg}; Beta ($\beta$) – \textit{Ins1, Ins2}; Delta ($\delta$) – \textit{Sst}; Gamma ($\gamma$) – \textit{Ppy}) and poly-hormonal sub-populations were annotated using a step-wise strategy of: \textbf{(i)} setting thresholds of hormone expression and \textbf{(ii)} identifying cells co-expressing any two hormone markers. Based on this, we were able to identify ‘Alpha-Delta’, ‘Alpha-Gamma’, ‘Beta-Gamma’ and ‘Delta-Gamma’ as the major poly-hormonal contributors. The other non-endocrine sub-poulations included: Endothelial (\textit{Plvap}), Mesenchymal (\textit{Pdgfra}), Schwann (\textit{Sox10}) and Ductal (\textit{Krt19}).\\\\The cells annotated as ‘Beta’ were then subset from the integrated endocrine data. We performed additional QC in order to exclude cells with \glslink{mrna}{mRNA} expression of other islet endocrine hormone markers (\textit{Gcg, Sst, Ppy, Pyy}). Next, we split the remaining $\beta$-cells into individual batches and re-integrated to obtain a $\beta$-cell specific subset on which further downstream analyses were performed.   


\subsubsection{\large Mapping of external datasets}
\label{sec:chp2_methods_labeltransfer}
In order to map: \textbf{(i)} a previously published dataset of islet macrophages in \gls{nod} mice \textbf{\cite{zakharov_single-cell_2020}} onto the integrated macrophages subset in this study and, \textbf{(ii)} a previous single-cell study of murine islets after 100 days of persistent hyperglycemia due to partial $\beta$-cell ablation \textbf{\cite{sachs_targeted_2020}} onto the integrated $\beta$-cell subset in this study, we utilized the label transfer workflow in Seurat. To perform data transfer, we first find anchors between the reference and query by projecting the \gls{pca} structure of the reference onto the query using the \textit{FindTransferAnchors()} function. After finding anchors, we used the \textit{TransferData()} function to re-annotate the macrophages and the $\beta$-cells in this study as per the datasets used for comparison. The newly transferred annotations were then visualized on \gls{umap} embeddings. 


\subsubsection{\large Compositional analysis}
The composition of the experimental groups by the different populations of immune \textbf{(Fig. \ref{fig:chp2_fullscRNA}B)} and endocrine \textbf{(Fig. \ref{fig:chp2_fullscRNA}C)}, and different sub-populations of macrophages \textbf{(Fig. \ref{fig:chp2_macs_composition})}, $\beta$-cells \textbf{(Fig. \ref{fig:chp2_betacells1}E)} and CD8\textsuperscript{+} T-cells \textbf{(Fig. \ref{fig:chp2_tcells2}B)} were computed as percentages of a given cell-type or sub-population across all cells in a particular group. The changes in cellular composition were analyzed using a mixed effects binomial model, comparing the number of cells with a certain sub-population between conditions, using the \textit{glmer()} function in lme4 package.

\subsubsection{\large Differential gene expression analysis}
%Prior to conducting differential analysis to identify genes across diet-feeding and aging, 
We performed an exploratory analysis to investigate whether the genes in individual sub-populations of macrophages and $\beta$-cells trended in the same direction across the groups. Specifically, we performed \gls{dge} analysis for every sub-population in macrophages (Macs-1, Macs-2, Macs-3 and Macs-5) and in $\beta$-cells (Beta-1, Beta-2 and Beta-3) for 1 week and 12 weeks of WD feeding and the aged cohorts compared to adult controls. We then visualized the average log-fold changes of the genes for every comparison, two sub-populations at a time in a `scatter plot'. All $\beta$-cell sub-populations depicted gene expression changes in a similar fashion across both time-points of WD feeding and aging. Therefore, we performed a global \gls{dge} analysis for all $\beta$-cells and identified a list of 521 \gls{de} genes across all the groups. These genes were further clustered into 10 \textit{k-means} clusters based on the aggregated expression across the five experimental groups. Similar analysis was also performed for Macs-2, whereby we identified a list of 895 \gls{de} genes across all groups, which were also clustered based on the aggregate gene expression across the groups. The differential analyses were performed with the \textit{run\textunderscore de()} function in LIBRA R package \textbf{\cite{}} which is a wrapper around the \textit{FindMarkers()} function in Seurat. The scatter plots were generated with custom scripts. Gene expression were aggregated across the groups using the \textit{AggregateExpression()} function with assays = ‘RNA’ and slot = ‘counts’. The \textit{k-means} clustering was performed with \textit{kmeans()} function in base R. The differential heatmap was generated with ComplexHeatmap R package.

\subsubsection{\large Gene ontology / Gene set enrichment analysis}

All \gls{go} term analysis for the sub-population markers and differential markers were performed using the R interface to the Enrichr database, via enrichR package. The input gene sets were compared against eight databases (GO \textunderscore Molecular \textunderscore Function \textunderscore2018, GO \textunderscore Cellular \textunderscore Component \textunderscore 2018, GO \textunderscore Biological\textunderscore Process\textunderscore 2018, KEGG\textunderscore 2019\textunderscore Human, MSigDB\textunderscore Hallmark\textunderscore 2020, Reactome\textunderscore 2016, WikiPathways\textunderscore 2019\textunderscore Human and BioCarta\textunderscore 2016). The enriched terms were further visualized using custom scripts.

\subsubsection{\large Differential Cell-Cell interaction analysis}
To infer how diet affects immune cell dynamics in the pancreatic islets, we performed differential \gls{lr} analysis between macrophage sub-populations (Macs-1, Macs-2, Macs-3 and Macs-5) and CD8\textsuperscript{+} T-cell sub-populations (Naive-CD8, Cytotoxic CD8, Memory-CD8 and IFN-responsive) using CellChat \textbf{\cite{jin_cellchat_2023}}. The raw counts for the indicated sub-populations were extracted from the corresponding Seurat objects. Features expressed in less than 10\% of the total cells were excluded from further analysis and the remaining features were converted to HGNC nomenclature using biomaRt package \textbf{\cite{}}.\\\\
\underline{\normalsize \textbf{Generation of individual CellChat objects}}\\\\
We first generated individual CellChat objects for each of the groups to be compared (Week-1 Chow, Week-1 WD, Weeks-12 Chow, Weeks-12 WD, Aging-Chow and Non-Aging-Chow). For every group, we first normalized the counts using the \textit{normalizeData()} function. The normalized counts were used to create a CellChat object using the \textit{createCellChat()} function and further processed using the standard pipeline. \textit{CellChatDB.human} was used as the \gls{lr} database and contains 1939 validated molecular interactions. Overexpressed ligands or receptors were identified in each sub-population with \textit{identifyOverExpressedGenes()} and further interactions were identified with \textit{identifyOverExpressedInteractions()}. Biologically significant cell-cell interactions were inferred using \textit{computeCommunProb()} function with parameters type = “truncatedMean” and trim set to 0.1. These interactions were further filtered using \textit{filterCommunication()} function with min.cells = 10. The dominant senders and receivers in every sub-population were identified using \textit{netAnalysis\textunderscore computeCentrality()} function.\\\\
\underline{\normalsize \textbf{Differential interaction analysis}}\\\\
We performed differential interaction analysis between WD and Chow at Week-1 and Week-12 time-points and between the aged and adult cohorts under Chow feeding by merging the corresponding CellChat objects. \textit{E.g.}, Week-1 Chow and Week-1 WD objects were merged into a single object using the \textit{mergeCellChat()} function. Before merging, the Week-1 Chow group was ‘lifted up’ to match the cell annotations of Week-1 WD group. Similarly, the Week-12 WD group was lifted to match Week-12 Chow group. This was accomplished using the \textit{liftCellChat()} function. Further downstream analysis of differential interactions between the groups was performed according to the relevant tutorials <insert ref> .

\clearpage


%\begin{comment}
    


\section{Computational methods for \autoref{chp:meta_analysis}}
\label{sec:chp3_methods}
%Experimental methods written by Julie Jerber, as in \cite{jerber2020population}.

\subsubsection{\large Datasets included in this study}

We comprehensively collected seven \gls{scr} datasets of pancreatic islets from several mouse models with $\beta$-cell decompensation and hyperglycemia. All datasets were generated with 10x Genomics Chromium technology. From the seven datasets, five were generated in-house as part of individual studies and were complemented with two previously published datasets. In addition to these, we utilized two additional datasets to demonstrate the usability of this integrated study and extend the findings to other models of $\beta$-cell workload. One of the two studies was generated and processed in-house. The raw data of the three publicly available studies were downloaded from European Nucleotide Archive (ENA). A brief description of these individual studies is outlined in the following section and is summarized in \textbf{\autoref{tab:app_chp3_study}}. All computational analyses of the \gls{scr} data were performed with Seurat v4 \textbf{\cite{hao_integrated_2021}}.\\

%\subsubsection{Datasets used for integration}}

\underline{\normalsize \textbf{Datasets used for integration}}\\


\textbf{Aging/Maturation\\}
In this physiological model, islets were isolated from wild-type male mice that were young (3-week-old) and aged (2-year-old) under standard housing conditions. The data from the 2-year-old mice was also utilized in the study of aging-associated inflammation of pancreatic islets in \textbf{\autoref{chp:diet_aging}}\\

\textbf{Meal Feeding\\}
In this study, we utilized wild-type BL/6 mice (male and female) in a time-restricted feeding paradigm, where food availability was restricted to the 12-hour dark phase, followed by a 12-hour fasting phase. On the final day of entrainment, subsequent to the fasting phase, one group received food for 4 hours (termed Fed group) and the other group underwent 4 hours of additional fasting (termed Fast group).\\

\textbf{Diet-induced obesity\\}
In this study, we provided adult male BL/6 mice (9-week-old) ad-libitum access to high-fat high-sucrose diet (HFHSD), termed as \acrfull{wd} for 1 week or 12 weeks and the corresponding controls were given normal chow diet for same duration. The physiological data showed that while acute \gls{wd}-feeding did not result in any significant weight gain, mice exhibited insulin hypersecretion at basal glucose levels, independently to insulin resistance. On the contrary, chronic \gls{wd}-feeding resulted in obesity and glucose intolerance, accompanied by insulin resistance. Additional details about this model can be found in \textbf{\autoref{chp:diet_aging}}.\\

\textbf{Mild genetic obesity\\}
In this model, the adaptive response of $\beta$-cells to mild obesity was studied in a leptin receptor (\textit{LepR}) deficient \textit{db/db} mouse model. This mouse model of \gls{t2d} is widely used and the leptin (\textit{Lep}) deficiency confers susceptibility to obesity, insulin resistance and \gls{t2d}. In this study, we used prediabetic \textit{db/db} 6-week-old mice (at the onset of hyperglycemia) and 9-week-old mice (developed hyperglycemia) and their corresponding controls, \textit{db/+} animals.\\

\textbf{Severe genetic obesity\\}
In this study, Chung \textit{et al.} performed \gls{scr} on islets isolated from wild-type mice and \textit{ob/ob} mice \textbf{\cite{chung_endocrine-exocrine_2020}}. The \textit{ob/ob} mice possess a recessive mutation in the gene encoding leptin (\textit{Lep}) hormone. As a result, the mice become severely obese along with hyperglycemia and hyperinsulinemia. The authors observed moderate overlap in $\beta$-cell populations between wild-type and \textit{ob/ob} mice, with the latter population depicting up-regulation of genes protein folding, translation and \glslink{er}{ER} stress.\\

\textbf{Insulin receptor blockade\\}
In this model of several hyperglycemia, wild-type mice were treated with S961, a peptide insulin receptor antagonist, for one week, thereby blocking insulin receptor (\textit{InsR}) signaling and impairing insulin-mediated effects, such as glucose uptake and metabolism, leading to insulin resistance and hyperglycemia.\\

\textbf{Partial $\beta$-cell ablation\\}
In this study, Sachs \textit{et al.} performed \gls{scr} on islets after 100 days of persistent hyperglycemia due to chemical $\beta$-cell ablation in the islet cell niche with \gls{stz} \textbf{\cite{sachs_targeted_2020}}. The remaining $\beta$-cells after ablation indicated \glslink{er}{ER} stress response and dysfunction, were dedifferentiated and involved a partial reversal to an embryonic or immature $\beta$-cell program.\\

%\subsubsection{\underline{Datasets used for validation}}
\underline{\normalsize \textbf{Datasets used for validation}}\\

\textbf{High-fat diet model\\}
In this study, Fu \textit{et al.} performed \gls{scr} on islets from a \gls{hfd}-induced mouse model of glucose intolerance \textbf{\cite{fu_single-cell_2023}}. The authors identified a \textit{CD81}\textsuperscript{high} sub-population which indicated an immature signature of $\beta$-cells compared with the \textit{CD81}\textsuperscript{low} sub-population, which had robust function.\\

\textbf{Unrestrained insulin secretion model\\}
In this study, we utilized the \textit{db/db} model with or without $\beta$-cell specific deletion of \textit{Lsd1} at the onset of (6-weeks) and during established (9-weeks) hyperglycemia. This allowed us to assess the function of \textit{Lsd1} in $\beta$-cell adaptation to insulin resistance. \textit{Lsd1} deletion in the lean state caused severe hypoglycemia, whereas, surprisngly, \textit{Lsd1} deletion on the \textit{db/db} background worsened the development of diabetes. This was associated with reduction in circulating insulin levels, thereby suggesting a $\beta$-cell defect underlying disease acceleration.\\ 

\subsubsection{\large Preprocessing of datasets}
\par The gene expression count matrices were generated for each experimental sample using the CellRanger analysis pipeline (v4.0.0), provided by 10x Genomics, and the \textit{mm10} (2020-A) genome build which is also provided by 10x Genomics. The filtered gene expression matrices, which excludes barcodes corresponding to background noise, were used for all subsequent downstream analyses. First, to reduce the effect of ambient expression, the ambient \glslink{mrna}{mRNA} contamination was removed using SoupX (v---) \textbf{\cite{young_soupx_2020}}. We estimated the contamination fraction based on the islet hormone and \acrfull{tf} markers: Alpha ($\alpha$-cells), \textit{Gcg, Arx, Ttr}; Beta ($\beta$-cells), \textit{Ins1, Ins2, Iapp}; Delta ($\delta$-cells), \textit{Sst} and Gamma (PP-cells), \textit{Ppy, Pyy}). The counts were then adjusted with \textit{adjustCounts()} function. Following ambient \glslink{mrna}{mRNA} correction, we filtered out doublets by applying Scrublet (v0.2.1) \textbf{\cite{wolock_scrublet_2019}}, on a per-sample basis, by manually adjusting the \textit{expected doublet rate} parameter depending on the number of cells obtained in that given sample.\\
\par The ambient \glslink{mrna}{mRNA} and doublet filtered matrices of individual samples were imported as Seurat objects with the \textit{CreateSeuratObject()} function, while filtering out: (i) features detected in less than 10 cells (\textit{min.cells = 10}) and (ii) cells with less than 500 features detected (\textit{min.features = 500}). The percentage of reads mapping to all mitochondrial genes (\textit{mt-}\textsuperscript{*}), L-ribosomal proteins (\textit{Rpl}\textsuperscript{*}), S-ribosomal proteins (\textit{Rps}\textsuperscript{*}) and dissociation-associated genes (DAGs, \hl{Supplementary Table XX}) were computed with the \textit{PercentageFeatureSet()} function. Next, for every individual sample, we filtered outlier cells based on \gls{umi} counts, feature counts and  with over 15\% of counts associated with mitochondrial genome.

\subsubsection{\large Data normalization and integration}
We performed study-wise aggregation of the filtered count matrices of the samples, in order to obtain seven study-specific, \glslink{qc}{QC}-performed count matrices. All further processing were performed on the level of individual studies. The matrices were log-normalized using \textit{NormalizeData()} function and 3000 \glspl{hvf} were selected using \textit{FindVariableFeatures()} function. To speed up the integration process, \gls{rpca} was applied, wherein, anchors between any two studies are determined by projecting each study into others’ \gls{pca} space. For this purpose, individual studies were scaled across the selected \glspl{hvf} using \textit{ScaleData()} function and \glspl{pc} were computed with \textit{RunPCA()} function. Following this, anchors between the studies were identified using \textit{FindIntegrationAnchors()} with parameter \textit{reduction = ‘rpca’}. The identified anchors were then used for integrating the studies via \textit{IntegrateData()} function. The integrated data was further scaled and centered across the \glspl{hvf}, and subjected to \gls{pc} analysis. The top 30 \glspl{pc} were further used to build a neighborhood graph with \textit{FindNeighbors(…, dims = 1:30, …)} function. The computed graph was used to: \textbf{(i)} identify clusters of cells using the original Louvain algorithm at a resolution parameter of 0.2, using the \textit{FindClusters(…, resolution = 0.2, …)} function, which yielded 16 clusters; \textbf{(ii)} further reduce the dimensionality of the integrated data from 30 \glspl{pc} to a two-dimensional (2D) projection via the \gls{umap} algorithm, implemented in the \textit{RunUMAP(…, min.dist = 0.1, …)} function.

\subsubsection{\large $\beta$-cell subset and re-integration}
\par We extracted cells annotated as Beta ($\beta$) and Proliferating-Endocrine from the full integrated dataset, in order to generate a $\beta$-cell specific subset. We performed additional \glslink{qc}{QC} in order to exclude cells with \glslink{mrna}{mRNA} expression of other islet endocrine hormone markers (\textit{Gcg, Sst, Ppy, Pyy}). Next, we split the remaining $\beta$-cells into individual studies and performed a study-level re-integration of the $\beta$-cells using the same streps as before. We identified $\beta$-cell specific \glspl{hvf} for the re-integration with \textit{SelectIntegrationFeatures()} function and using the variance stabilizing transformation (VST) method. In VST, the estimation of gene expression mean and regularized variance is based on sample mean and variance. After re-integration, the $\beta$-cells were scaled and centered, and we performed \gls{pca} and computed \gls{umap} embedding.\\
\par We grouped the $\beta$-cell specific \glspl{hvf} into modules of co-regulated genes with the Monocle v3 package, using the \textit{find\textunderscore gene\textunderscore modules()} function. Next, we aggregated the expression of all genes in every module across all $\beta$-cells in every experimental group and performed hierarchical clustering of the groups based on the aggregated expression of the modules, using base R. Heatmap depicting the aggregated expression of the modules across the experimental groups was generated using ComplexHeatmap R package.\\

\par We identified $\beta$-cell subsets using the \textit{FindClusters(…, resolution = 0.1, …)} function, which yielded 6 subsets. For $\beta$-1, $\beta$-2 and $\beta$-3 subsets, we identified subset-specific markers by performing pair-wise comparison of the three subsets, using the \textit{FindMarkers()} function. We used the \textit{FindAllMarkers()} function to identify subset markers for $\beta$-4, $\beta$-5 and $\beta$-6 subsets. Using the subset-specific markers, we identified functional categories from the \gls{kegg} and Reactome categories with Metascape \textbf{\cite{zhou_metascape_2019}}. The composition of the groups with the $\beta$-cell subsets was computed as the percentage of cells identified as a particular subset over all the cells from a particular group and depicted in bar plots with custom plotting scripts.

\subsubsection{\large \gls{pca} of non-proliferating $\beta$-cells}
To focus on the downstream analysis of non-proliferating $\beta$-cells, we excluded the $\beta$-4 Proliferating subset and recomputed the \glspl{pc} for the remaining subsets using the \textit{RunPCA()} function in Seurat. In order to rank the $\beta$-cells by the first two \glspl{pc} (\gls{pc}1 and \gls{pc}2) separately, we utilized the \textit{rank()} function in base R. Next, we performed \textit{Min-Max} normalization of the \gls{pc}-based ranks to the range of [0,1]. Histogram plots depicting the density of $\beta$-cells grouped by subsets across \gls{pc}1 \hl{(Fig. )} and \gls{pc}2 \hl{(Fig. )} ranks were generated using custom plotting scripts. The list of features with the strongest contribution to  \gls{pc}1 and \gls{pc}2 were generated using the \textit{TopFeatures()} function in Seurat. Heatmaps depicting the scaled expression of the top identified features across $\beta$-cells ordered by their \gls{pc}1 \hl{(Fig. )} and \gls{pc}2 \hl{(Fig. )} ranks were generated using Seurat. The top features for \gls{pc}1 and \gls{pc}2 were also used to identify functional categories from the \gls{kegg} and Reactome categories with Metascape \textbf{\cite{zhou_metascape_2019}} \hl{(Figs. )}.\\\\
The composite \gls{pc} representing the Maturtiy-Workload axis of non-proliferating $\beta$-cells was obtained by summing \gls{pc}1 and \gls{pc}2. Prior to summation, the \glspl{pc} were multiplied by their corresponding variance:
\begin{equation}
    \text{Composite \gls{pc}\textsubscript{1,2}} = (\sigma^2_{PC1} \times PC1) + (\sigma^2_{PC2} \times PC2)
\end{equation}
where $\sigma^2$ represents the variance of the \glspl{pc}.\\\\
The cells were then ranked and the ranks of the composite \gls{pc} were normalized in the same fashion as the individual components, and the density of $\beta$-cells grouped according to the subsets across this composite \gls{pc} were visualized as a histogram plot using custom plotting scripts. 


\subsubsection{\large Gene regulatory network inference and regulon module analysis}
To infer \glspl{grn}: (i) for the integrated subset of Alpha-Beta cell-types, and (ii) for non-proliferating $\beta$-cells, we utilized pySCENIC v0.11.2 on a high-performance computing cluster. We extracted the raw counts across all the \glspl{hvg} for both subsets into matrices. Alongside a list of 1721 mouse \glspl{tf}, this gene expression matrices served as input for calculating gene co-expression modules via GRNBoost2, using \textit{pyscenic grn}. The \gls{tf}/target associations and their corresponding importance metric (IM) values were used as input for RCisTarget alongside and ranking databases for motifs in the promoter of the genes [up to 500 \gls{bp} upstream of the \gls{tss}], 10 kb (±5 kb) and 20 kb (±10 kb) around the \gls{tss}, using \textit{pyscenic ctx}, in order to remove indirect gene targets lacking \textit{cis}-regulatory motifs associated with the \gls{tf}. The remaining co-expressed TF-target genes are then grouped into regulons. The activity of these regulons was computed with AUCell, using \textit{pyscenic aucell}, which calculates the enrichment of a regulon using \gls{auc} of the genes that define the regulon. These activity data were further binarized (assigned an ON or OFF value, per regulon, per cell) by thresholding on the AUC values of the given regulon, using the \textit{binarize()} function.\\\\
Regulon modules for the non-proliferating $\beta$-cells were identified based on the \gls{csi} (Fuxman Bass et al., 2013), which is a context-dependent measure for identifying specific associating partners. To compute \gls{csi}, first, the \gls{pcc} of activity scores is evaluated for each pair of regulons. Next, for a fixed pair of regulons (A and B), the corresponding \gls{csi} is defined as the fraction of regulons whose \gls{pcc} with A and B is lower than the \gls{pcc} between A and B. Hierarchical clustering with complete linkage was performed based on \gls{csi} matrix to identify five regulon modules.\\


Heatmap depicting the binary regulon matrix for Alpha-Beta subset \hl{(Fig.)} were generated using the ComplexHeatmap R package by randomly sampling 30000 cells from the subset. Heatmap depicting the \gls{csi} of the five regulon modules identified for non-proliferating $\beta$-cells \hl{(Fig.)} were also generated with ComplexHeatmap.  Network plot \hl{(Fig.)} were created from Cytoscape from top 1\%  of the TF/target associated from coexpression modules based on the reported importance metric for non-proliferating $\beta$-cells. The betweenness centrality of the nodes in the network were computed within Cytoscape with the NetworkAnalyzer module. The mean activity of regulons across the subsets and the groups were computed using \textit{aggregate()} function with parameter `FUN=mean' in base R. Dot plots depicting the mean regulon activity were generated with custom plotting scripts.  \hl{<TFs heatmap across PC1 \& PC2>}


\subsubsection{\large Gene-set scoring}
We computed scores of: \textbf{(i)} the gene modules identified for pseudobulk $\beta$-cells across experimental groups, and \textbf{(ii)} the gene-sets upregulated in human T2D from MIA, by performing gene-set scoring using the \textit{AddModuleScore()} function in Seurat. In case of latter, we first extracted genes for the gene-sets of interest using the \textit{msigdb\textunderscore gsets()} function from the msigdbr package. The scores are the average expression levels of each gene-set on single cell level, subtracted by the aggregated expression of control feature sets. All analyzed genes in a given gene-set are binned based on averaged expression, and the control features are randomly selected from each bin. The gene-set scores were visualized with violin plots using the \textit{VlnPlot()} function from Seurat and with custom plotting scripts.

\subsubsection{\large Computing fold-changes of genes}
To compute the fold-changes of genes associated with gene-sets upregulated in human T2D from MIA between $\beta$-cell workload groups and their corresponding controls in each study, we utilized the \textit{FoldChange()} function in Seurat. The function returns the log fold change and the difference in percentages of cells expressing each gene for different groups. The dot plots depicting the fold changes were generated with custom plotting scripts. 

\subsubsection{\large Validation analyses via query mapping}
In order to map the integrated queries onto the integrated non-proliferating $\beta$-cell reference subset, we utilized the label transfer workflow in Seurat. To perform data transfer, we first find anchors between the reference and query by projecting the \gls{pca} structure of the reference onto the query using the \textit{FindTransferAnchors()} function with parameters, \textit{dims = 1:20} and \textit{reference.reduction = `pca'}. After finding anchors, we used the \textit{TransferData()} function to classify the cells from the query dataset based on the annotations of the reference subset. Following this, the subset composition of the groups in the query was computed as the percentage of cells that were predicted to be a particular subset (the annotations which were transferred from the reference) over all the cells from a particular group. The compositions were depicted in bar plots with custom plotting scripts.


%\end{comment}