%!TEX root = ../thesis.tex
%******************************
%	 Acronyms 
%*****************************


\newacronym[]{atp}{ATP}{adenosine triphosphate}
\newacronym[]{adp}{ADP}{adenosine diphosphate}
\newacronym[]{ampk}{AMPK}{5' adenosine monophosphate-activated protein kinase}
\newacronym{katp}{K\textsubscript{ATP}}{ATP-sensitive K\textsuperscript{+}}
\newacronym[]{auc}{AUC}{area under the curve}
\newacronym{anova}{ANOVA}{analysis of variance}
\newacronym{ap1}{AP-1}{activator protein 1}

\newacronym{bp}{bp}{base pairs}
\newacronym[plural=BCRs,firstplural=B-Cell receptors]{bcr}{BCR}{B-cell receptor}

\newacronym[]{csi}{CSI}{connection specificity index}
\newacronym[plural=CCIs,firstplural=cell-cell interactions]{cci}{CCI}{cell-cell interaction}
\newacronym[plural=CCCs,firstplural=cell-cell communications]{ccc}{CCC}{cell-cell communication}
\newacronym[]{camp}{cAMP}{cyclic adenosine monophosphate}

\newacronym{wnt}{WNT/Wnt}{Wingless Type}

\newacronym{cytof}{CyTOF}{cytometry by time-of-flight}
\newacronym{crispr}{CRISPR}{Clustered Regularly Interspaced Short Palindromic Repeats}
\newacronym{cite}{CITE-seq}{Cellular Indexing of Transcriptomes and Epitopes by sequencing}
\newacronym{cdna}{cDNA}{complementary DNA}


\newacronym[plural=DAGs,firstplural=dissociation-associated genes]{dag}{DAG}{dissociation-associated gene}
\newacronym{dge}{DGE}{differential gene expression}
\newacronym{de}{DE}{differentially  expressed}
\newacronym{dr}{DR}{Dimensionality reduction}
\newacronym{im}{IM}{importance metric}
% \newacronym{damp}{DAMPs}{}
\newacronym[plural=DAMPs,firstplural=danger-associated molecular patterns]{damp}{DAMP}{danger-associated molecular pattern}
\newacronym[plural=MAMPs,firstplural=metabolism-associated molecular patterns]{mamp}{MAMP}{metabolism-associated molecular pattern}
\newacronym[plural=TLRs,firstplural=Toll-like receptors]{tlr}{TLR}{Toll-like receptor}
\newacronym[plural=PRRs,firstplural=pattern recognition receptors]{prr}{PRR}{pattern recognition receptor}

\newacronym[plural=NLRs,firstplural=Nucleotide-binding oligomerization domain-like receptors]{nlr}{NLR}{Nucleotide-binding oligomerization domain-like receptor}
\newacronym{mapk}{MAPK}{Mitogen-activated protein kinase}
\newacronym{nlrp3}{NLRP3}{Nucleotide-Binding Domain, Leucine-Rich–Containing Family, Pyrin Domain–Containing-3}



\newacronym{er}{ER}{endoplasmic reticulum}
\newacronym{erad}{ERAD}{\glslink{er}{ER}-associated protein degradation}

\newacronym[plural=FAs, firstplural=fatty acids]{fa}{FA}{fatty acid}
\newacronym[plural=FFAs, firstplural=free fatty acids]{ffa}{FFA}{free fatty acid}
\newacronym[]{fret}{FRET}{Fluorescence Resonance Energy Transfer}
\newacronym[]{ffpe}{FFPE}{formalin-fixed paraffin embedded}
\newacronym{fc}{FC}{fold change}
\newacronym[plural=pDCs,firstplural=plasmacytoid dendritic cells]{pdc}{pDCs}{plasmacytoid dendritic cells}
\newacronym{go}{GO}{gene ontology}
\newacronym{gsea}{GSEA}{gene set enrichment analysis}
\newacronym[plural=GRNs,firstplural=gene regulatory networks]{grn}{GRN}{gene regulatory network}
\newacronym[]{gck}{GCK / \textit{Gck}}{glucokinase}
\newacronym[]{glut}{GLUT}{glucose transporters}
\newacronym[]{g6p}{G6P}{glucose-6-phosphate}
\newacronym[]{gsis}{GSIS}{glucose-stimulated insulin secretion}
\newacronym{gip}{GIP}{Glucose-dependent insulinotropic polypeptide}
\newacronym{glp1}{GLP-1}{glucagon-like peptide 1}
\newacronym{glp1ra}{GLP-1RA}{\gls{glp1} receptor agonists}
\newacronym[plural=GEMs]{gem}{GEM}{Gel Bead in Emulsion}
\newacronym{gwa}{GWA}{genome-wide association}
\newacronym{gandt}{G\&T-seq}{single-cell genome and transcriptome sequencing}

\newacronym{hfd}{HFD}{high-fat diet}
\newacronym{hfhsd}{HFHSD}{high-fat high-sucrose diet}
\newacronym[plural=HVFs,firstplural=highly variable features]{hvf}{HVF}{highly variable feature}
\newacronym[plural=HVGs,firstplural=highly variable genes]{hvg}{HVG}{highly variable gene}
\newacronym{hande}{H\&E}{Hematoxylin and Eosin}

\newacronym{idf}{IDF}{International Diabetes Federation}
\newacronym{imc}{IMC}{Imaging Mass Cytometry}
\newacronym{insr}{InsR}{insulin receptor}
\newacronym[plural=ILCs,firstplural=Innate Lymphoid cells]{ilc}{ILC}{Innate Lymphoid cell}
\newacronym{ihc}{IHC}{immunohistochemistry}
\newacronym{if}{IF}{Immunofluorescence}
\newacronym[plural=IFNs, firstplural = interferons]{ifn}{IFN}{interferon}

\newacronym{kb}{kb}{kilobase (pair)}
\newacronym{kegg}{KEGG}{Kyoto Encyclopedia of Genes and Genome}
\newacronym{knn}{\textit{k}NN}{\textit{k-} nearest neighbors}
\newacronym{dpp4i}{DPP-4i}{dipeptidyl peptidase 4 inhibitors}

\newacronym{lr}{L-R}{ligand-receptor}
\newacronym{lri}{LRI}{ligand-receptor interaction}
\newacronym[plural=LCFAs, firstplural=long-chain fatty acids]{lcfa}{LCFA}{long-chain fatty acid}
\newacronym{il}{IL}{Interleukin}

\newacronym[]{mtor}{mTORC1}{mammalian target of rapamycin complex 1}
\newacronym{mrna}{mRNA}{messenger RNA}
\newacronym{mia}{MIA}{Mouse Islet Atlas}
\newacronym[plural=NEFAs, firstplural=non-esterified fatty acids]{nefa}{NEFA}{non-esterified fatty acid}
\newacronym[]{nadph}{NADPH}{nicotinamide adenine dinucleotide phosphate hydrogen}
\newacronym[]{mhc2}{MHC-II}{major histocompatibility complex class II}
%\newacronym{mtor}{mTORC1}{mammalian target of rapamycin complex 1}

\newacronym{nk}{NK}{Natural Killer}


\newacronym{otiff}{OME-TIFF}{Open Microscopy Environment-Tag Image File Format}
\newacronym{oxphos}{OxPhos}{oxidative phosphorylation}

\newacronym{pcc}{PCC}{pearson correlation coefficient}
\newacronym[plural=PCs, firstplural=principal components (PCs)]{pc}{PC}{principal component}
\newacronym{pca}{PCA}{principal component analysis}
\newacronym{paga}{PAGA}{partition-based graph abstraction}
\newacronym{pdac}{PDAC}{pancreatic ductal adenocarcinoma}
\newacronym{pp}{PP}{pancreatic polypeptide}
\newacronym{perk}{PERK}{protein kinase R-like \glsentryshort{er} kinase}
\newacronym{pcp}{PCP}{planar cell polarity}


\newacronym{qc}{QC}{quality control}
\newacronym{vst}{VST}{variance stabilizing transformation}

\newacronym[]{rrp}{RRP}{readily releasable pool}
\newacronym{rpca}{RPCA}{reciprocal PCA}
\newacronym[]{rp}{RP}{reserve pool}
\newacronym{rnaseq}{RNA-seq}{RNA sequencing}
\newacronym{roc}{ROC}{receiver operating characteristics}
\newacronym{rt}{RT}{reverse transcription}
\newacronym{ros}{ROS}{reactive oxygen species}
\newacronym[plural=ROIs, firstplural=regions of interest]{roi}{ROI}{region of interest}
\newacronym{treg}{T-regs}{regulatory T-cells}

\newacronym[plural=SGs,firstplural=secretory granules]{sg}{SG}{secretory granule}
\newacronym{scr}{scRNA-seq}{single-cell RNA sequencing}
\newacronym{scatac}{scATAC-seq}{single-cell assay for transposase-accessible chromatin using sequencing}
\newacronym{snatac}{snATAC-seq}{single-nuclei assay for transposase-accessible chromatin using sequencing}
\newacronym{scbs}{scBS-seq}{single-cell bisulfite sequencing}
\newacronym{chip}{scChIP-seq}{single-cell chromatin immunoprecipitation sequencing}
\newacronym{hiapp}{hIAPP}{human islet amyloid polypeptide}
\newacronym{iapp}{IAPP}{islet amyloid polypeptide}

\newacronym{stz}{STZ}{Streptozotocin}
\newacronym{st}{ST}{spatial transcriptomics}
\newacronym[plural=TCRs,firstplural=T-Cell receptors]{tcr}{TCR}{T-cell receptor}
\newacronym{mandt}{scM\&T-seq}{single-cell methylome and transcriptome sequencing}
\newacronym{stat}{STAT}{signal transducer and activator of transcription}
\newacronym{stat1}{STAT1}{signal transducer and activator of transcription 1}
\newacronym{stat3}{STAT3}{signal transducer and activator of transcription 3}

\newacronym[]{tsne}{tSNE}{t-distributed stochastic neighbour embedding}
\newacronym{ti}{TI}{trajectory inference}
\newacronym[plural=TFs,firstplural=transcription factors (TFs)]{tf}{TF}{transcription factor}
\newacronym{tss}{TSS}{transcription start site}
\newacronym{t1d}{T1D}{Type 1 diabetes}
\newacronym{t2d}{T2D}{Type 2 diabetes}
\newacronym{tiff}{TIFF}{Tag Image File Format}
\newacronym[plural=TMAs, firstplural=tissue microarrays]{tma}{TMA}{tissue microarray}
\newacronym{tnf}{TNF}{tumor necrosis factor}
% \newacronym{nfkb}{NF-$\kappa$B}{nuclear factor kappa-light-chain-enhancer of activated B-cells}
\newacronym{nfkb}{NF-$\kappa$B}{Nuclear factor-$\kappa$B}
\newacronym{nod}{NOD}{nonobese diabetic}

\newacronym[]{umap}{UMAP}{uniform manifold and approximation projection}
\newacronym[]{upr}{UPR}{unfolded protein response}

\newacronym[plural=VDCCs,firstplural=voltage-dependent L-type calcium channels]{vdcc}{VDCC}{voltage-dependent L-type calcium channel}
\newacronym[]{wat}{WAT}{white adipose tissues}
\newacronym[]{wd}{WD}{western diet}
\newacronym{fltp1}{FLTP}{Flattop}
\newacronym{fltp2}{\textit{Fltp}}{Flattop}

%

\newacronym{sglt2i}{SGLT2i}{sodium-glucose co-transporter 2 inhibitors}
\newacronym[plural=NSAIDs,firstplural=non-steroidal anti-inflammatory drugs]{nsaid}{NSAID}{non-steroidal anti-inflammatory drug}

% \newacronym{chipseq}{ChIP-seq}{Chromatin immuno-precipitation sequencing}
% \newacronym{CD/CV}{CD/CV}{Common disease / common variant}
% \newacronym{cpm}{CPM}{counts per million (reads)}
% \newacronym{cv}{CV}{coefficient of variation}
% \newacronym{da}{DA}{dopaminergic neurons}
% \newacronym{defendo}{defendo}{definitive endoderm}
% \newacronym{dna}{DNA}{deoxyribonucleic acid} 
% \newacronym{dof}{dof}{degrees of freedom}
% \newacronym{ebi}{EBI}{European Bioinformatics Institute}
% \newacronym[plural=eQTL, firstplural=expression quantitative trait loci (eQTL)]{eqtl}{eQTL}{expression quantitative trait locus}
% \newacronym{es}{ES}{embryonic stem}
% \newacronym[plural=ESCs, firstplural=embryonic stem cells (ESCs)]{esc}{ESC}{embryonic stem cell}
\newacronym{facs}{FACS}{fluorescence-activated cell sorting}
% \newacronym{fdr}{FDR}{false discovery rate}
% \newacronym{gxe}{GxE}{genotype-environment}
% \newacronym{glm}{GLM}{generalised linear model}
% \newacronym{gtex}{GTEx}{genotype-tissue expression}

% \newacronym[plural=hESCs, firstplural=human embryonic stem cells (hESCs)]{hesc}{hESC}{human embryonic stem cell}
% \newacronym{hgp}{HGP}{Human Genome Project}
% \newacronym{hipsci}{HipSci}{human induced pluripotent stem cell initiative}
% \newacronym{hvgs}{HVGs}{highly variable genes}
% \newacronym{ips}{iPS}{induced pluripotent stem}
% \newacronym[plural=iPSCs, firstplural=induced pluripotent stem cells (iPSCs)]{ipsc}{iPSC}{induced pluripotent stem cell}
% \newacronym{ivf}{IVF}{\textit{in vitro} fertilisation}

% \newacronym{ld}{LD}{linkage disequilibrium}
% \newacronym[plural=LMMs, firstplural=linear mixed models (LMMs)]{lmm}{LMM}{linear mixed model}
% \newacronym{lrt}{LRT}{likelihood ratio test}
% \newacronym{maf}{MAF}{minor allele frequency}
% \newacronym{mesendo}{mesendo}{mesendoderm}
% \newacronym{ml}{ML}{maximum likelihood}
% \newacronym{mle}{MLE}{maximum likelihood estimator}
% \newacronym{mqtl}{mQTL}{(DNA) methylation QTL}

% \newacronym{ngs}{NGS}{next generation sequencing}
% \newacronym{nhgri}{NHGRI}{National Human Genome Research Institute}
% \newacronym{ols}{OLS}{ordinary least squares}
% \newacronym{pbmc}{PBMC}{peripheral blood mononuclear cell}

% \newacronym{qq}{QQ}{quantile-quantile}
% \newacronym{qtl}{QTL}{quantitative trait locus}
% \newacronym{qtls}{QTL}{quantitative trait loci}
% \newacronym{scrnaseq}{scRNA-seq}{single-cell RNA sequencing} 
\newacronym{rna}{RNA}{ribonucleic acid}
\newacronym{dna}{DNA}{deoxyribonucleic acid}
% \newacronym{rnaseq}{RNA-seq}{RNA sequencing}
% \newacronym{scnt}{SCNT}{somatic-cell nuclear transfer}
% \newacronym[plural=SNPs, firstplural=single nucleotide polymorphisms (SNPs)]{snp}{SNP}{single nucleotide polymorphism}
% \newacronym{tf}{TF}{transcription factor}
% \newacronym{trna}{tRNA}{transfer RNA}
\newacronym[plural=UMIs, firstplural=unique molecular identifiers (UMIs)]{umi}{UMI}{unique molecular identifier}