
%!TEX root = ../thesis.tex
% ******************************* Thesis Appendix B ********************************
\begin{comment}


\clearpage

\section{Supplementary Note 1}
\label{sec:suppnote1}
\subsubsection{Doublet Detection}

\clearpage

\section{Supplementary Note 2}
\label{sec:suppnote2}
\subsubsection{Ambient mRNA contamination}

During library preparation in droplet-based single-cell technologies, cells that are stressed or undergoing apoptosis, may release their mRNA content into the cell suspension. These freely floating transcripts in the background are referred to as \textbf{‘ambient mRNA’} and can cross-contaminate the endogenous gene expression profiles of cells, thereby complicating downstream analysis steps, such as cell-type annotation \textbf{\cite{yang_decontamination_2020}}. Apart from this, high background mRNA counts could also occur when working with specific tissues \textbf{\cite{10x_genomics_introduction_nodate,madissoon_scrna-seq_2019}}; while employing nuclei isolation protocols which typically release cytoplasmic RNA into the solution \textbf{\cite{10x_genomics_introduction_nodate}}; from abundant cell-types whose RNA can contaminate the less abundant cell-types \textbf{\cite{10x_genomics_introduction_nodate,caglayan_neuronal_2022}} or from other exogeneous sources of contamination \textbf{\cite{10x_genomics_introduction_nodate}}. Therefore, accounting for background RNA profiles in scRNA-seq data is necessary to ensure the reliability and accuracy of results of downstream data analysis. Several computational tools have been developed to identify and remove ambient mRNA \textbf{\cite{yang_decontamination_2020,young_soupx_2020,fleming_unsupervised_2023,yan_emptynn_2021,alvarez_enhancing_2020,muskovic_dropletqc_2021}}. These tools work to either remove entire droplets based on their corresponding expression profiles or remove the ambient mRNA associated with barcodes.\footnote{Ultimately, the use of background correction tools depend on the extent of the contamination as well as the overall goals of the experiment. It is recommended to carefully inspect the data beforehand as not every dataset might require ambient mRNA correction.\\}\\\\
% \vspace{3mm}
% \begin{Abstract}
% \vspace{3mm}
% The following section provides a descriptive explanation of SoupX \textbf{\cite{young_soupx_2020}}, which is one of the several aforementioned computational tools for the estimation and correction of ambient mRNA. In \textbf{Chapter 2} and \textbf{Chapter 3}, I have used SoupX as part of the pre-processing workflow in order to account for background contamination. 
% \vspace{3mm}
% \end{Abstract}
% \vspace{3mm}
SoupX is a computational method that quantifies the extent of ambient mRNA contamination and recovers true, cell-specific signal. The authors refer to the input solution with the floating cell free mRNAs as \textbf{‘the soup’}. The method consists of three steps:
\begin{enumerate}
    \item Estimate the composition of the soup.
    \item Compute the \st{cell-specific} contamination fraction ($\mathbf{\rho}$) and
    \item Generate a corrected count matrix after removing the soup-based expression.
\end{enumerate}
SoupX estimates the profile the the soup from the empty droplets in the scRNA-seq data. In an 10x Chromium experiment, this corresponds to the \textbf{‘unfiltered feature-barcode’} matrix file. The expression profile of the soup is automatically estimated from the empty droplets and the counts data. In cases, when the empty droplets file might be unavailable, the profile of the soup can be estimated manually. An important intermediate step is to provide the clustering information for each sample which renders the subsequent correction step more robust. Next, the level of background contamination ($\mathbf{\rho}$) in each sample is estimated and can be accomplished in two ways: \textbf{the automatic method to estimate the contamination fraction} or \textbf{the manual method, by providing a list of ‘non-expressed’ genes}. Skipping these methods, the contamination fraction can also be set directly at a fixed value.\footnote{As SoupX is designed to be conservative, manually setting a global contamination fraction can remove a greater proportion of the background with a low risk of losing true biological signals.}\\\\
For the automatic estimation, SoupX makes use of highly-expressed cluster-specific markers to estimate $\mathbf{\rho}$ for every gene, against the clusters in which it can be confidently said to not be expressed. Using the estimates of these gene-cluster pairs, SoupX calculates a posterior distribution, using a Poisson-likelihood, of the contamination fraction and determines its best estimate. Therefore, to get a better estimate of $\mathbf{\rho}$, it is imperative to have, for every set of cells, a list of genes that is known to not be expressed in that set and by measuring the expression of those genes, the global contamination fraction can be estimated.
\clearpage

\section{Supplementary Note 3}
\label{sec:suppnote3}

\subsubsection{Cell-Cell Interactions}


To infer CCIs from scRNA-seq data, CellChat makes use of \textbf{CellChatDB}, a database of literature supported LRIs, in both mouse and human. CellChatDB is specific as it contains LRIs involving multimeric receptors, cofactor molecules as well as co-stimulatory or co-inhibitory membrane bound receptors, and is grouped into several signalling pathway families. In addition, CellChatDB allows for updating of the database by adding user-defined ligand-receptor pairs. Next, to infer intercellular communications from scRNA-seq data, CellChat first identifies differential signalling genes across all cell-types in the data using the Wilcoxon rank sum test, and for these differential genes, computes the \textbf{“ensemble average expression”} in the cell-type. Following this, a communication probability value for every LRI pair is modelled based on the \textbf{Law of Mass Action} and a \textbf{Hill function}. By using the expression levels of ligands and receptors as “proxies” for their concentration and modelling interactions with expression-based Hill functions \footnote{In CellChat, the Hill function is used to incorporate the effects of positive and negative effectors into the framework, thereby providing a more nuanced and accurate model of cell-cell communication}. In addition, the frequencies of the cell-types are also accounted for in the probability calculation. Finally, significant interactions between two cell-types are identified on the basis of a random permutation test <add footnote here>   . Using centrality metrics from graph theory, CellChat identifies higher order information from a weighted-directed network, wherein the computed communication probabilities are used as weights. This allows for the identification of dominant signalling sources as well as mediators and influencers within the network. CellChat can also predict key outgoing and incoming signals for specific cell-types as well as coordinated responses amongst different cell-types by pattern recognition approaches based on non-negative matrix factorization. An important application of CellChat is the joint analyses of two or more datasets by constructing a shared manifold space using a shared nearest neighbour (SNN) similarity network. This enables the identification of conserved as well as context-specific interactions between the datasets being compared.    
\end{comment}

\chapter{R packages and Python libraries for \glsentryshort{scr} Analysis}
\label{chp:app_packages}

\verbatiminput{Appendix3/sessionInfo.txt}